<!DOCTYPE html>
<!--[if lte IE 8 ]><html lang="en" class="no-js oldIE"><![endif]-->
<!--[if (gt IE 8)|!(IE)]><!--><html lang="en" class="no-js"><!--<![endif]-->
<head>
    <title>A mechanistic stress model of protein evolution accounts for site-specific evolutionary rates and their relationship with packing density and flexibility | BMC Evolutionary Biology | Full Text</title>
    
    
    <meta name="journal_id" content='12862'/>
    
    <meta name="dc.title" content='A mechanistic stress model of protein evolution accounts for site-specific evolutionary rates and their relationship with packing density and flexibility'/>
    
    <meta name="dc.description" content='Protein sites evolve at different rates due to functional and biophysical constraints. It is usually considered that the main structural determinant of a site&#8217;s rate of evolution is its Relative Solvent Accessibility (RSA). However, a recent comparative study has shown that the main structural determinant is the site&#8217;s Local Packing Density (LPD). LPD is related with dynamical flexibility, which has also been shown to correlate with sequence variability. Our purpose is to investigate the mechanism that connects a site&#8217;s LPD with its rate of evolution. We consider two models: an empirical Flexibility Model and a mechanistic Stress Model. The Flexibility Model postulates a linear increase of site-specific rate of evolution with dynamical flexibility. The Stress Model, introduced here, models mutations as random perturbations of the protein&#8217;s potential energy landscape, for which we use simple Elastic Network Models (ENMs). To account for natural selection we assume a single active conformation and use basic statistical physics to derive a linear relationship between site-specific evolutionary rates and the local stress of the mutant&#8217;s active conformation. We compare both models on a large and diverse dataset of enzymes. In a protein-by-protein study we found that the Stress Model outperforms the Flexibility Model for most proteins. Pooling all proteins together we show that the Stress Model is strongly supported by the total weight of evidence. Moreover, it accounts for the observed nonlinear dependence of sequence variability on flexibility. Finally, when mutational stress is controlled for, there is very little remaining correlation between sequence variability and dynamical flexibility. We developed a mechanistic Stress Model of evolution according to which the rate of evolution of a site is predicted to depend linearly on the local mutational stress of the active conformation. Such local stress is proportional to LPD, so that this model explains the relationship between LPD and evolutionary rate. Moreover, the model also accounts for the nonlinear dependence between evolutionary rate and dynamical flexibility.'/>
    
    <meta name="dc.source" content='BMC Evolutionary Biology 2014 14:1'/>
    
    <meta name="dc.format" content='text/html'/>
    
    <meta name="dc.publisher" content='BioMed Central'/>
    
    <meta name="dc.date" content='2014-04-09'/>
    
    <meta name="dc.type" content='OriginalPaper'/>
    
    <meta name="dc.language" content='En'/>
    
    <meta name="dc.copyright" content='2014 Huang et al.; licensee BioMed Central Ltd.'/>
    
    <meta name="dc.rights" content='http://creativecommons.org/licenses/by/2.0'/>
    
    <meta name="dc.rightsAgent" content='reprints@biomedcentral.com'/>
    
    <meta name="prism.issn" content='1471-2148'/>
    
    <meta name="prism.publicationName" content='BMC Evolutionary Biology'/>
    
    <meta name="prism.publicationDate" content='2014-04-09'/>
    
    <meta name="prism.volume" content='14'/>
    
    <meta name="prism.number" content='1'/>
    
    <meta name="prism.section" content='OriginalPaper'/>
    
    <meta name="prism.startingPage" content='78'/>
    
    <meta name="prism.copyright" content='2014 Huang et al.; licensee BioMed Central Ltd.'/>
    
    <meta name="prism.rightsAgent" content='reprints@biomedcentral.com'/>
    
    <meta name="prism.url" content='http://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-14-78'/>
    
    <meta name="citation_pdf_url" content='http://bmcevolbiol.biomedcentral.com/track/pdf/10.1186/1471-2148-14-78?site=bmcevolbiol.biomedcentral.com'/>
    
    <meta name="citation_abstract_html_url" content='http://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-14-78'/>
    
    <meta name="citation_fulltext_html_url" content='http://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-14-78'/>
    
    <meta name="citation_journal_title" content='BMC Evolutionary Biology'/>
    
    <meta name="citation_publisher" content='BioMed Central'/>
    
    <meta name="citation_issn" content='1471-2148'/>
    
    <meta name="citation_title" content='A mechanistic stress model of protein evolution accounts for site-specific evolutionary rates and their relationship with packing density and flexibility'/>
    
    <meta name="citation_volume" content='14'/>
    
    <meta name="citation_issue" content='1'/>
    
    <meta name="citation_date" content='2014/04/09'/>
    
    <meta name="citation_publication_date" content='2014/04/09'/>
    
    <meta name="citation_firstpage" content='78'/>
    
    <meta name="citation_fulltext_world_readable" content=''/>
    
    <meta name="dc.creator" content='Tsun-Tsao Huang'/>
    
    <meta name="dc.creator" content='Mar&#237;a Laura del Valle Marcos'/>
    
    <meta name="dc.creator" content='Jenn-Kang Hwang'/>
    
    <meta name="dc.creator" content='Julian Echave'/>
    
    <meta name="dc.subject" content='Evolutionary Biology'/>
    
    <meta name="dc.subject" content='Animal Systematics/Taxonomy/Biogeography'/>
    
    <meta name="dc.subject" content='Entomology'/>
    
    <meta name="dc.subject" content='Genetics and Population Dynamics'/>
    
    <meta name="dc.subject" content='Life Sciences, general'/>
    
    <meta name="citation_author" content='Tsun-Tsao Huang'/>
    
    <meta name="citation_author_institution" content='Institute of Bioinformatics and Systems Biology, National Chiao Tung University, HsinChu, Taiwan'/>
    
    <meta name="citation_author_institution" content='Center for Bioinformatics Research, National Chiao Tung University, HsinChu, Taiwan'/>
    
    <meta name="citation_author" content='Mar&#237;a Laura del Valle Marcos'/>
    
    <meta name="citation_author_institution" content='Escuela de Ciencia y Tecnolog&#237;a, Universidad Nacional de San Mart&#237;n, Buenos Aires, Argentina'/>
    
    <meta name="citation_author" content='Jenn-Kang Hwang'/>
    
    <meta name="citation_author_institution" content='Institute of Bioinformatics and Systems Biology, National Chiao Tung University, HsinChu, Taiwan'/>
    
    <meta name="citation_author_institution" content='Center for Bioinformatics Research, National Chiao Tung University, HsinChu, Taiwan'/>
    
    <meta name="citation_author" content='Julian Echave'/>
    
    <meta name="citation_author_institution" content='Escuela de Ciencia y Tecnolog&#237;a, Universidad Nacional de San Mart&#237;n, Buenos Aires, Argentina'/>
    
    <meta name="citation_reference" content='citation_journal_title=Nat Rev Genet; citation_title=An integrated view of protein evolution; citation_author=C Pal; citation_author=B Papp; citation_author=MJ Lercher; citation_volume=7; citation_issue=5; citation_publication_date=2006; citation_pages=337-348; citation_doi=10.1038/nrg1838; citation_id=CR1;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Curr Opin Struct Biol; citation_title=Protein evolution constraints and model-based techniques to study them; citation_author=JL Thorne; citation_volume=17; citation_issue=3; citation_publication_date=2007; citation_pages=337-341; citation_doi=10.1016/j.sbi.2007.05.006; citation_id=CR2;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Nat Rev Mol Cell Biol; citation_title=Structural and functional constraints in the evolution of protein families; citation_author=CL Worth; citation_author=S Gong; citation_author=TL Blundell; citation_volume=10; citation_issue=10; citation_publication_date=2009; citation_pages=709-720; citation_id=CR3;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Curr Opin Struct Biol; citation_title=Signatures of protein biophysics in coding sequence evolution; citation_author=CO Wilke; citation_author=DA Drummond; citation_volume=20; citation_issue=3; citation_publication_date=2010; citation_pages=385-389; citation_doi=10.1016/j.sbi.2010.03.004; citation_id=CR4;'/>
    
    <meta name="citation_reference" content='citation_journal_title=BMC Evol Biol; citation_title=Biophysical and structural considerations for protein sequence evolution; citation_author=JA Grahnen; citation_author=P Nandakumar; citation_author=J Kubelka; citation_author=DA Liberles; citation_volume=11; citation_issue=1; citation_publication_date=2011; citation_pages=361; citation_doi=10.1186/1471-2148-11-361; citation_id=CR5;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Protein Sci; citation_title=The interface of protein structure, protein biophysics, and molecular evolution; citation_author=DA Liberles; citation_author=SA Teichmann; citation_author=I Bahar; citation_author=U Bastolla; citation_author=J Bloom; citation_author=E Bornberg-Bauer; citation_author=LJ Colwell; citation_author=AP de Koning; citation_author=NV Dokholyan; citation_author=J Echave; citation_author=A Elofsson; citation_author=DL Gerloff; citation_author=RA Goldstein; citation_author=JA Grahnen; citation_author=MT Holder; citation_author=C Lakner; citation_author=N Lartillot; citation_author=SC Lovell; citation_author=G Naylor; citation_author=T Perica; citation_author=DD Pollock; citation_author=T Pupko; citation_author=L Regan; citation_author=A Roger; citation_author=N Rubinstein; citation_author=E Shakhnovich; citation_author=K Sjolander; citation_author=S Sunyaev; citation_author=AI Teufel; citation_author=JL Thorne; citation_volume=21; citation_issue=6; citation_publication_date=2012; citation_pages=769-785; citation_doi=10.1002/pro.2071; citation_id=CR6;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Mol Biol Evol; citation_title=Solvent accessibility and purifying selection within proteins of Escherichia coli and Salmonella enterica; citation_author=CD Bustamante; citation_author=JP Townsend; citation_author=DL Hartl; citation_volume=17; citation_issue=2; citation_publication_date=2000; citation_pages=301-308; citation_doi=10.1093/oxfordjournals.molbev.a026310; citation_id=CR7;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Mol Biol Evol; citation_title=The pattern of amino acid replacements in alpha/beta-barrels; citation_author=AM Dean; citation_author=C Neuhauser; citation_author=E Grenier; citation_author=GB Golding; citation_volume=19; citation_issue=11; citation_publication_date=2002; citation_pages=1846-1864; citation_doi=10.1093/oxfordjournals.molbev.a004009; citation_id=CR8;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Mol Biol Evol; citation_title=Structural determinants of protein evolution are context-sensitive at the residue level; citation_author=EA Franzosa; citation_author=Y Xia; citation_volume=26; citation_issue=10; citation_publication_date=2009; citation_pages=2387-2395; citation_doi=10.1093/molbev/msp146; citation_id=CR9;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Genetics; citation_title=The relationship between relative solvent accessibility and evolutionary rate in protein evolution; citation_author=DC Ramsey; citation_author=MP Scherrer; citation_author=T Zhou; citation_author=CO Wilke; citation_volume=188; citation_issue=2; citation_publication_date=2011; citation_pages=479-488; citation_doi=10.1534/genetics.111.128025; citation_id=CR10;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Mol Biol Evol; citation_title=Site-specific structural constraints on protein sequence evolutionary divergence: local packing density versus solvent exposure; citation_author=S-W Yeh; citation_author=J-W Liu; citation_author=S-H Yu; citation_author=C-H Shih; citation_author=J-K Hwang; citation_author=J Echave; citation_volume=31; citation_issue=1; citation_publication_date=2014; citation_pages=135-139; citation_doi=10.1093/molbev/mst178; citation_id=CR11;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Flexibility and packing in proteins; citation_author=B Halle; citation_volume=99; citation_issue=3; citation_publication_date=2002; citation_pages=1274-1279; citation_doi=10.1073/pnas.032522499; citation_id=CR12;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Mol Biol Evol; citation_title=Sequence evolution correlates with structural dynamics; citation_author=Y Liu; citation_author=I Bahar; citation_volume=29; citation_issue=9; citation_publication_date=2012; citation_pages=2253-2263; citation_doi=10.1093/molbev/mss097; citation_id=CR13;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Protein Eng Des Sel; citation_title=Protein sequence entropy is closely related to packing density and hydrophobicity; citation_author=H Liao; citation_author=W Yeh; citation_author=D Chiang; citation_author=RL Jernigan; citation_author=B Lustig; citation_volume=18; citation_issue=2; citation_publication_date=2005; citation_pages=59-64; citation_doi=10.1093/protein/gzi009; citation_id=CR14;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Proteins; citation_title=Evolutionary information hidden in a single protein structure; citation_author=C-H Shih; citation_author=C-M Chang; citation_author=Y-S Lin; citation_author=W-C Lo; citation_author=J-K Hwang; citation_volume=80; citation_issue=6; citation_publication_date=2012; citation_pages=1647-1657; citation_doi=10.1002/prot.24058; citation_id=CR15;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Evol Appl; citation_title=Structural dynamics flexibility informs function and evolution at a proteome scale; citation_author=Z Nevin Gerek; citation_author=S Kumar; citation_author=S Banu Ozkan; citation_volume=6; citation_issue=3; citation_publication_date=2013; citation_pages=423-433; citation_doi=10.1111/eva.12052; citation_id=CR16;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Annu Rev Biophys; citation_title=Global dynamics of proteins: bridging between structure and function; citation_author=I Bahar; citation_author=TR Lezon; citation_author=LW Yang; citation_author=E Eyal; citation_volume=39; citation_publication_date=2010; citation_pages=23-42; citation_doi=10.1146/annurev.biophys.093008.131258; citation_id=CR17;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Phys Life Rev; citation_title=Comparing proteins by their internal dynamics: exploring structure&#8211;function relationships beyond static structural alignments; citation_author=C Micheletti; citation_volume=10; citation_issue=1; citation_publication_date=2013; citation_pages=1-26; citation_doi=10.1016/j.plrev.2012.10.009; citation_id=CR18;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Methods Mol Biol; citation_title=Elastic network models: theoretical and empirical foundations; citation_author=YH Sanejouand; citation_volume=924; citation_publication_date=2013; citation_pages=601-616; citation_doi=10.1007/978-1-62703-017-5_23; citation_id=CR19;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Biophys J; citation_title=Anisotropy of fluctuation dynamics of proteins with an elastic network model; citation_author=AR Atilgan; citation_author=SR Durell; citation_author=RL Jernigan; citation_author=MC Demirel; citation_author=O Keskin; citation_author=I Bahar; citation_volume=80; citation_issue=1; citation_publication_date=2001; citation_pages=505-515; citation_doi=10.1016/S0006-3495(01)76033-X; citation_id=CR20;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Chem Phys Lett; citation_title=Evolutionary divergence of protein structure: the linearly forced elastic network model; citation_author=J Echave; citation_volume=457; citation_issue=4&#8211;6; citation_publication_date=2008; citation_pages=413-416; citation_doi=10.1016/j.cplett.2008.04.042; citation_id=CR21;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Proteins; citation_title=A perturbative view of protein structural variation; citation_author=J Echave; citation_author=FM Fernandez; citation_volume=78; citation_issue=1; citation_publication_date=2010; citation_pages=173-180; citation_doi=10.1002/prot.22553; citation_id=CR22;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Pure Appl Chem; citation_title=Why are the low-energy protein normal modes evolutionarily conserved?; citation_author=J Echave; citation_volume=84; citation_issue=9; citation_publication_date=2012; citation_pages=1931-1937; citation_doi=10.1351/PAC-CON-12-02-15; citation_id=CR23;'/>
    
    <meta name="citation_reference" content='citation_journal_title=J Chem Theory Comput; citation_title=Evaluation of protein elastic network models based on an analysis of collective motions; citation_author=E Fuglebakk; citation_author=N Reuter; citation_author=K Hinsen; citation_volume=9; citation_issue=12; citation_publication_date=2013; citation_pages=5618-5628; citation_doi=10.1021/ct400399x; citation_id=CR24;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Protein elastic network models and the ranges of cooperativity; citation_author=L Yang; citation_author=G Song; citation_author=RL Jernigan; citation_volume=106; citation_issue=30; citation_publication_date=2009; citation_pages=12347-12352; citation_doi=10.1073/pnas.0902159106; citation_id=CR25;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Proteins; citation_title=Deriving protein dynamical properties from weighted protein contact number; citation_author=C-P Lin; citation_author=S-W Huang; citation_author=Y-L Lai; citation_author=S-C Yen; citation_author=C-H Shih; citation_author=C-H Lu; citation_author=C-C Huang; citation_author=J-K Hwang; citation_volume=72; citation_issue=3; citation_publication_date=2008; citation_pages=929-935; citation_doi=10.1002/prot.21983; citation_id=CR26;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Bioinformatics; citation_title=Rate4Site: an algorithmic tool for the identification of functional regions in proteins by surface mapping of evolutionary determinants within their homologues; citation_author=T Pupko; citation_author=RE Bell; citation_author=I Mayrose; citation_author=F Glaser; citation_author=N Ben-Tal; citation_volume=18; citation_issue=Suppl 1; citation_publication_date=2002; citation_pages=S71-S77; citation_doi=10.1093/bioinformatics/18.suppl_1.S71; citation_id=CR27;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Mol Biol Evol; citation_title=Comparison of site-specific rate-inference methods for protein sequences: empirical Bayesian methods are superior; citation_author=I Mayrose; citation_author=D Graur; citation_author=N Ben-Tal; citation_author=T Pupko; citation_volume=21; citation_issue=9; citation_publication_date=2004; citation_pages=1781-1791; citation_doi=10.1093/molbev/msh194; citation_id=CR28;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Psychon Bull Rev; citation_title=AIC model selection using Akaike weights; citation_author=EJ Wagenmakers; citation_author=S Farrell; citation_volume=11; citation_issue=1; citation_publication_date=2004; citation_pages=192-196; citation_doi=10.3758/BF03206482; citation_id=CR29;'/>
    
    <meta name="citation_reference" content='citation_journal_title=BMC Pharmacol; citation_title=An evaluation of R2 as an inadequate measure for nonlinear models in pharmacological and biochemical research: a Monte Carlo approach; citation_author=AN Spiess; citation_author=N Neumeyer; citation_volume=10; citation_publication_date=2010; citation_pages=6; citation_doi=10.1186/1471-2210-10-6; citation_id=CR30;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Gene; citation_title=Evolutionary conservation of protein vibrational dynamics; citation_author=S Maguid; citation_author=S Fernandez-Alberti; citation_author=J Echave; citation_volume=422; citation_issue=1&#8211;2; citation_publication_date=2008; citation_pages=7-13; citation_doi=10.1016/j.gene.2008.06.002; citation_id=CR31;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Methods Mol Biol; citation_title=Packing regularities in biological structures relate to their dynamics; citation_author=RL Jernigan; citation_author=A Kloczkowski; citation_volume=350; citation_publication_date=2007; citation_pages=251-276; citation_id=CR32;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Bioinformatics; citation_title=Site-specific evolutionary rates in proteins are better modeled as non-independent and strictly relative; citation_author=AD Fernandes; citation_author=WR Atchley; citation_volume=24; citation_issue=19; citation_publication_date=2008; citation_pages=2177-2183; citation_doi=10.1093/bioinformatics/btn395; citation_id=CR33;'/>
    
    <meta name="prism.doi" content='10.1186/1471-2148-14-78'/>
    
    <meta name="dc.identifier" content='10.1186/1471-2148-14-78'/>
    
    <meta name="citation_doi" content='10.1186/1471-2148-14-78'/>
    
    <meta name="DOI" content='10.1186/1471-2148-14-78'/>
    

    <meta charset="UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=edge"/>
<meta name="viewport" content="width=device-width, initial-scale=1"/>

    <link rel="apple-touch-icon" sizes="57x57"
          href="/static/images/favicons/biomedcentral/apple-touch-icon-57x57-e249e00d61.png"/>
    <link rel="apple-touch-icon" sizes="60x60"
          href="/static/images/favicons/biomedcentral/apple-touch-icon-60x60-fb82a36f32.png"/>
    <link rel="apple-touch-icon" sizes="72x72"
          href="/static/images/favicons/biomedcentral/apple-touch-icon-72x72-e911dece4c.png"/>
    <link rel="apple-touch-icon" sizes="76x76"
          href="/static/images/favicons/biomedcentral/apple-touch-icon-76x76-63bd2b8bd3.png"/>
    <link rel="apple-touch-icon" sizes="114x114"
          href="/static/images/favicons/biomedcentral/apple-touch-icon-114x114-00d3da59e2.png"/>
    <link rel="apple-touch-icon" sizes="120x120"
          href="/static/images/favicons/biomedcentral/apple-touch-icon-120x120-189ccdbff3.png"/>
    <link rel="apple-touch-icon" sizes="144x144"
          href="/static/images/favicons/biomedcentral/apple-touch-icon-144x144-4b231fd999.png"/>
    <link rel="apple-touch-icon" sizes="152x152"
          href="/static/images/favicons/biomedcentral/apple-touch-icon-152x152-852d14549a.png"/>
    <link rel="apple-touch-icon" sizes="180x180"
          href="/static/images/favicons/biomedcentral/apple-touch-icon-180x180-2cfd7af96a.png"/>
    <link rel="icon" type="image/png"
          href="/static/images/favicons/biomedcentral/android-chrome-192x192-408ad37d82.png" sizes="192x192"/>
    <link rel="icon" type="image/png" href="/static/images/favicons/biomedcentral/favicon-16x16-51659b4472.png"
          sizes="16x16"/>
    <link rel="icon" type="image/png" href="/static/images/favicons/biomedcentral/favicon-32x32-893c91359a.png"
          sizes="32x32"/>
    <link rel="icon" type="image/png" href="/static/images/favicons/biomedcentral/favicon-96x96-08ca36c9e5.png"
          sizes="96x96"/>
    <link rel="icon" type="image/png" href="/static/images/favicons/biomedcentral/favicon-194x194-0969f3a317.png"
          sizes="194x194"/>
    <link rel="manifest" href="/static/images/favicons/biomedcentral/manifest.json"/>
    <link rel="shortcut icon" href="/static/images/favicons/biomedcentral/favicon-dc10a51d5f.ico"/>
    <meta name="msapplication-TileColor" content="#da532c"/>
    <meta name="msapplication-config"
          content="/static/images/favicons/biomedcentral/browserconfig.xml"/>
    <meta name="msapplication-TileImage"
          content="/static/images/favicons/biomedcentral/mstile-144x144-8b117caf89.png"/>
    <meta name="theme-color" content="#ffffff"/>

<script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
<script>(function(d) {if (sessionStorage.getItem('webfonts') == 'true'){d.className += ' webfonts-loaded';}})(document.documentElement);</script>

    <link rel="stylesheet" media="screen" href="/static/styles/basic/basic-2d50733a26.css"/>
    <link rel="stylesheet" media="print" href="/static/styles/biomedcentral/biomedcentral-print-b286681189.css"/>
    
    <link rel="stylesheet" class="js-ctm" href="/static/styles/biomedcentral/biomedcentral-f033a5eaec.css"
          media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">


<script>var _gaq = [];</script>

    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'bmcevolbiol.biomedcentral.com',
            siteWithPath: 'bmcevolbiol.biomedcentral.com' + window.location.pathname,
            twitterHashtag: 'bmcevolbiol',
            cmsPrefix: 'http://studio-cms.springer.com/studio/',
            
            doi: '10.1186/1471-2148-14-78',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>


    <script>
        window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;

        (function() {
            var sampleRate = 20;

            
                ga('create', "UA-62065396-10", {
                    cookieDomain: 'none',
                    allowLinker: true,
                    siteSpeedSampleRate: sampleRate
                });

                ga("set", "anonymizeIp", true);
                ga('send', 'pageview');
            

            ga('create', "UA-54492316-9", {
                cookieDomain: 'none',
                allowLinker: true,
                siteSpeedSampleRate: sampleRate,
                name: 'global'
            });

            ga("global.set", "anonymizeIp", true);
            ga('global.send', 'pageview');
            ga('set', 'transport', 'beacon');
        }())
    </script>
    <script async src='https://www.google-analytics.com/analytics.js'></script>


    
    <link rel="canonical" href="//bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-14-78"/>
    


    
    <meta property="og:url" content="http://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-14-78"/>
    <meta property="og:type" content="article"/>
    <meta property="og:site_name" content="BMC Evolutionary Biology"/>
    <meta property="og:title" content="A mechanistic stress model of protein evolution accounts for site-specific evolutionary rates and their relationship with packing density and flexibility"/>
    <meta property="og:description" content="Protein sites evolve at different rates due to functional and biophysical constraints. It is usually considered that the main structural determinant of a site’s rate of evolution is its Relative Solvent Accessibility (RSA). However, a recent comparative study has shown that the main structural determinant is the site’s Local Packing Density (LPD). LPD is related with dynamical flexibility, which has also been shown to correlate with sequence variability. Our purpose is to investigate the mechanism that connects a site’s LPD with its rate of evolution. We consider two models: an empirical Flexibility Model and a mechanistic Stress Model. The Flexibility Model postulates a linear increase of site-specific rate of evolution with dynamical flexibility. The Stress Model, introduced here, models mutations as random perturbations of the protein’s potential energy landscape, for which we use simple Elastic Network Models (ENMs). To account for natural selection we assume a single active conformation and use basic statistical physics to derive a linear relationship between site-specific evolutionary rates and the local stress of the mutant’s active conformation. We compare both models on a large and diverse dataset of enzymes. In a protein-by-protein study we found that the Stress Model outperforms the Flexibility Model for most proteins. Pooling all proteins together we show that the Stress Model is strongly supported by the total weight of evidence. Moreover, it accounts for the observed nonlinear dependence of sequence variability on flexibility. Finally, when mutational stress is controlled for, there is very little remaining correlation between sequence variability and dynamical flexibility. We developed a mechanistic Stress Model of evolution according to which the rate of evolution of a site is predicted to depend linearly on the local mutational stress of the active conformation. Such local stress is proportional to LPD, so that this model explains the relationship between LPD and evolutionary rate. Moreover, the model also accounts for the nonlinear dependence between evolutionary rate and dynamical flexibility."/>
    <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/12862.jpg"/>

</head>
<body class="fulltext">
    <div class="container" style="height: 0"><div id="wt_refpoint"></div></div>
    <p class="skip-to-main">
    <a class="skip-to-main__link" href="#main">Skip to main content</a>
</p>

    <div id="top">
        
<div id="RealMediaLeaderboard" class="adsbox RealMediaAd Ad Ad--leaderboard" data-position="Top">
    <div class="Ad_content">
        <p class="Ad_label">Advertisement</p>
        
    </div>
</div>


         
            <header class="header" role="banner">
                <div class="header__inner">
                    <a id="header-logo" href="//www.biomedcentral.com/" class="logo header__logo" title="BioMed Central">
                        <img srcset="/static/images/biomed-central-logo-2d9caecd7b.svg" src="/static/images/biomed-central-logo-5300b47908.png" class="logo__img" alt="Biomed Central" />
                    </a>
                    <div class="header__right">
                        <div class="toggles js-fade-toggle">
                            <p id="loginLink" class="sign-in toggles__item" data-replace="true" data-placeholder="/placeholder/v1/username">
    <a class="sign-in__link" href="//www.biomedcentral.com/my-account" data-track-event="click" data-event-category="Account" data-event-action="Click My Account">
        <span class="sign-in__no-js-account-hide">Login<span class="sign-in__js-account-hide"> to </span></span><span class="sign-in__js-account-hide sign-in__no-js-capitalise">your account</span>
    </a>
</p>
                            
                            <a href="#globalSearch" class="toggles__item toggles__item--search" data-toggle="search"><span class="sr-only">Search</span></a>
                        </div>
                        <form role="search" id="globalSearch" class="search search--expandable header__search" method="GET" action="//www.biomedcentral.com/search" data-track-event="submit"
      data-event-category="Search and Results" data-event-action="Submit search" data-event-label="">
    <div class="search__content">
        <label for="publisherSearch" class="sr-only">Search BioMed Central articles</label>
        <input id="publisherSearch" class="search__input" autocomplete="off" role="textbox" name="query" type="text" value="" title="Search BioMed Central" placeholder="Search BioMed Central articles"/>
        <button id="search" class="search__button" type="submit">
            <span class="sr-only">Search</span>
        </button>
        <input type="hidden" name="searchType" value="publisherSearch"/>
    </div>
</form>

                    </div>
                    
                </div>
            </header>
            <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>
        
        <main role="main">
            
            <header class="journal-header" role="banner">
                <div class="journal-header__inner">
                    <h2 class="journal-header__title journal-header__title--medium" id="journalTitle">
                        
                        <a class="journal-header__title-text" href="/">
                            BMC Evolutionary Biology
                        </a>
                    </h2>
                    
                </div>
                <div class="navbar">
                    <div class="navbar__inner">
                        

                        
                        <nav class="navbar__nav">
                            <h3 class="navbar__menu-toggle" data-toggle="collapse" data-target="journal-menu">
                                <span class="sr-only">Main </span>menu
                            </h3>
                            <ul id="journal-menu" class="navbar__menu" id="siteNavigation" role="menu">
                                
                                    <li class="navbar__item" role="menuitem">
                                        <a class="navbar__link" href='/'>Home</a>
                                    </li>
                                
                                    <li class="navbar__item" role="menuitem">
                                        <a class="navbar__link" href='/about'>About</a>
                                    </li>
                                
                                    <li class="navbar__item" role="menuitem">
                                        <a class="navbar__link is-active" href='/articles'>Articles</a>
                                    </li>
                                
                                    <li class="navbar__item" role="menuitem">
                                        <a class="navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                    </li>
                                
                            </ul>
                        </nav>
                        
                    </div>
                </div>
                <div class="journal-identity journal-identity--networks-aqua">
                    
                </div>
            </header>
            
        
            <div class="container">
                
                <div id="main" class="layout">
                
                    <div class="layout__main--wide" id="main-content">
                        <div class="block" id="Test-ImgSrc">
                            
                            
    <div class="FulltextWrapper">
        <div class="Categories">
            <div class="ArticleCategory">Research article</div>
            <div class="OpenAccessLabel">Open Access</div>
            
        </div>
        
    </div>

                            <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" lang="en">A mechanistic stress model of protein evolution accounts for site-specific evolutionary rates and their relationship with packing density and flexibility</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames"><ul class="u-listReset"><li class="Author" data-jumpto="Aff"><span xmlns="http://www.w3.org/1999/xhtml" class="AuthorName">Tsun-Tsao&nbsp;Huang</span><span class="EqualContributionSymbol">†</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author" data-jumpto="Aff"><span xmlns="http://www.w3.org/1999/xhtml" class="AuthorName">María&nbsp;Laura&nbsp;del Valle Marcos</span><span class="EqualContributionSymbol">†</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup>, </li><li class="Author" data-jumpto="Aff"><span xmlns="http://www.w3.org/1999/xhtml" class="AuthorName">Jenn-Kang&nbsp;Hwang</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup> and </li><li class="Author" data-jumpto="Aff"><span xmlns="http://www.w3.org/1999/xhtml" class="AuthorName">Julian&nbsp;Echave</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup><a href="mailto:julian.echave@unsam.edu.ar" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul><div class="AffiliationLegend"><div class="AffiliationLegendLine"><span class="EqualContributionSymbol">†</span>Contributed equally</div></div></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">BMC Evolutionary Biology</span><span class="ArticleCitation_Year">2014</span><span class="ArticleCitation_Volume"><strong>14</strong>:78</span></span></div><p class="ArticleDOI"><strong>DOI: </strong>10.1186/1471-2148-14-78</p><p class="Copyright">© &nbsp;Huang et al.; licensee BioMed Central Ltd.&nbsp;2014</p></div></div><div class="ArticleHistory"><p class="History HistoryReceived"><strong>Received: </strong>16&nbsp;January&nbsp;2014</p><p class="History HistoryAccepted"><strong>Accepted: </strong>21&nbsp;March&nbsp;2014</p><p class="History HistoryOnlineDate"><strong>Published: </strong>9&nbsp;April&nbsp;2014</p></div></div>
                            
                            
                            
    

                            <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract" id="Abs1" lang="en"><h2 class="Heading js-ToggleCollapseSection">Abstract</h2><div class="js-CollapseSection"><div xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p class="Para">Protein sites evolve at different rates due to functional and biophysical constraints. It is usually considered that the main structural determinant of a site’s rate of evolution is its Relative Solvent Accessibility (RSA). However, a recent comparative study has shown that the main structural determinant is the site’s Local Packing Density (LPD). LPD is related with dynamical flexibility, which has also been shown to correlate with sequence variability. Our purpose is to investigate the mechanism that connects a site’s LPD with its rate of evolution.</p>
              </div><div xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p class="Para">We consider two models: an empirical Flexibility Model and a mechanistic Stress Model. The Flexibility Model postulates a linear increase of site-specific rate of evolution with dynamical flexibility. The Stress Model, introduced here, models mutations as random perturbations of the protein’s potential energy landscape, for which we use simple Elastic Network Models (ENMs). To account for natural selection we assume a single active conformation and use basic statistical physics to derive a linear relationship between site-specific evolutionary rates and the local stress of the mutant’s active conformation.</p>
                <p class="Para">We compare both models on a large and diverse dataset of enzymes. In a protein-by-protein study we found that the Stress Model outperforms the Flexibility Model for most proteins. Pooling all proteins together we show that the Stress Model is strongly supported by the total weight of evidence. Moreover, it accounts for the observed nonlinear dependence of sequence variability on flexibility. Finally, when mutational stress is controlled for, there is very little remaining correlation between sequence variability and dynamical flexibility.</p>
              </div><div xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p class="Para">We developed a mechanistic Stress Model of evolution according to which the rate of evolution of a site is predicted to depend linearly on the local mutational stress of the active conformation. Such local stress is proportional to LPD, so that this model explains the relationship between LPD and evolutionary rate. Moreover, the model also accounts for the nonlinear dependence between evolutionary rate and dynamical flexibility.</p>
              </div><div xmlns="http://www.w3.org/1999/xhtml" class="KeywordGroup" lang="en">
              <h3 xmlns="" class="Heading">Keywords</h3>
              <span class="Keyword">Protein evolution</span>
              <span class="Keyword">Site-specific substitution rate</span>
              <span class="Keyword">Local packing density</span>
              <span class="Keyword">Elastic network model</span>
              <span class="Keyword">Flexibility</span>
              <span class="Keyword">Stress</span>
              <span class="Keyword">Mean square fluctuation</span>
              <span class="Keyword">Mean local mutational stress</span>
            </div></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1"><h2 class="Heading js-ToggleCollapseSection">Background</h2><div class="js-CollapseSection"><p xmlns="http://www.w3.org/1999/xhtml" class="Para">Due to functional and biophysical constraints, different protein sites evolve at different rates of amino-acid substitution [<span class="CitationRef"><a href="#CR1">1</a></span>–<span class="CitationRef"><a href="#CR6">6</a></span>]. The most popular structural correlate of a site’s substitution rate is its Relative Solvent Accessibility (RSA) [<span class="CitationRef"><a href="#CR7">7</a></span>–<span class="CitationRef"><a href="#CR10">10</a></span>]. In a thorough assessment of many structural properties as predictors of site-specific rates of evolution, Franzosa and Xia showed that the only two with significant independent contributions are RSA and CN, the Contact Number, with RSA performing slightly better [<span class="CitationRef"><a href="#CR9">9</a></span>]. However, in a more recent study, Yeh et al. compared RSA with two Local Packing Density (LPD) measures, CN and the Weighted Contact Number (WCN), and found that both LPD measures correlate better than RSA with evolutionary rates [<span class="CitationRef"><a href="#CR11">11</a></span>]. Moreover, they found that once LPD is controlled for, the independent contribution of RSA is small. Thus, LPD seems to be the main structural determinant of rate of evolution at site level. The purpose of the present work is to study possible mechanisms that connect LPD to evolutionary rates.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">A possible link could be dynamical flexibility. A site’s flexibility, quantified by its Mean Squared Fluctuation (MSF), is approximately proportional to 1/LPD [<span class="CitationRef"><a href="#CR12">12</a></span>]. A flexibility-based explanation assumes that a site’s rate of evolution increases with its dynamical flexibility. Within this framework 1/LPD would be just a “proxy” of a site’s flexibility, which would be the actual determinant of its evolutionary rate. Such interpretation would seem to be supported by empirical correlation studies of sequence variability vs. MSF [<span class="CitationRef"><a href="#CR13">13</a></span>] and variability vs. 1/LPD [<span class="CitationRef"><a href="#CR14">14</a></span>, <span class="CitationRef"><a href="#CR15">15</a></span>], and by a recent study based on a different dynamical flexibility measure [<span class="CitationRef"><a href="#CR16">16</a></span>]. Such a flexibility-based explanation not only makes some intuitive sense, but it is attractive because it is in line with the increasing acknowledgement of the role of dynamics for protein function [<span class="CitationRef"><a href="#CR17">17</a></span>, <span class="CitationRef"><a href="#CR18">18</a></span>]. Therefore, we postulate as our null model an explicit empirical Flexibility Model according to which a site’s rate of evolution depends linearly on its MSF.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">The main drawback of the previous flexibility-based interpretations, and the empirical Flexibility Model we set up to make their underlying assumptions explicit, is that no mechanism is proposed. To this end, here we propose a mechanistic alternative model. We model mutations as random perturbations of the parameters of the protein’s potential energy landscape and natural selection as a function of the probability that a mutant adopts a specific active conformation. Using basic statistical physics and certain simplifying assumptions, we derive that according to this model a site’s evolutionary rate will depend on the local stress introduced in the active structure by mutating it. Therefore, we shall call it the Stress Model.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">We will show that the Stress Model explains both the dependence of site-specific rates of evolution on packing density and on dynamical flexibility in terms of the local stress introduced by mutations on the protein’s active structure.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1"><h2 class="Heading js-ToggleCollapseSection">Methods</h2><div class="js-CollapseSection"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Elastic network models</h3>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Let the conformation of an N-sites protein be represented by the column vector of the 3&nbsp;N Cartesian coordinates of its N <em xmlns="" class="EmphasisTypeItalic">C</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">α</em>
                  </sub> atoms: <span xmlns="" class="EmphasisTypeBoldItalic">r</span> = (<em xmlns="" class="EmphasisTypeItalic">x</em><sub>1</sub>&nbsp;<em xmlns="" class="EmphasisTypeItalic">y</em><sub>1</sub>&nbsp;<em xmlns="" class="EmphasisTypeItalic">z</em><sub>1</sub>&nbsp;<em xmlns="" class="EmphasisTypeItalic">x</em><sub>2</sub>&nbsp;<em xmlns="" class="EmphasisTypeItalic">y</em><sub>2</sub>&nbsp;<em xmlns="" class="EmphasisTypeItalic">z</em><sub>2</sub> … <em xmlns="" class="EmphasisTypeItalic">x</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">N</em>
                  </sub>&nbsp;<em xmlns="" class="EmphasisTypeItalic">y</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">N</em>
                  </sub>&nbsp;<em xmlns="" class="EmphasisTypeItalic">z</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">N</em>
                  </sub>&nbsp;)<sup>
                    <em xmlns="" class="EmphasisTypeItalic">T</em>
                  </sup>. <span xmlns="" class="EmphasisTypeBoldItalic">r</span><sub>
                    <span xmlns="" class="EmphasisTypeBoldItalic">i</span>
                  </sub> = (<em xmlns="" class="EmphasisTypeItalic">x</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>&nbsp;<em xmlns="" class="EmphasisTypeItalic">y</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>&nbsp;<em xmlns="" class="EmphasisTypeItalic">z</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>)<sup>
                    <em xmlns="" class="EmphasisTypeItalic">T</em>
                  </sup> is the position vector of the <em xmlns="" class="EmphasisTypeItalic">i</em>th <em xmlns="" class="EmphasisTypeItalic">C</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">α</em>
                  </sub>. The vector joining sites <em xmlns="" class="EmphasisTypeItalic">i</em>&nbsp; and <em xmlns="" class="EmphasisTypeItalic">j</em> is <span xmlns="" class="EmphasisTypeBoldItalic">d</span><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub> = <span xmlns="" class="EmphasisTypeBoldItalic">r</span><sub>
                    <em xmlns="" class="EmphasisTypeItalic">j</em>
                  </sub> - <span xmlns="" class="EmphasisTypeBoldItalic">r</span><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub> with length <em xmlns="" class="EmphasisTypeItalic">d</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub> = <span xmlns="" class="EmphasisTypeBoldItalic">d</span><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub>. We use <span xmlns="" class="EmphasisTypeBoldItalic">r</span><sup>0</sup> for the protein’s equilibrium conformation in which the <em xmlns="" class="EmphasisTypeItalic">i</em>th site is at <span class="InlineEquation" id="IEq1"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mi mathvariant="bold-italic">r</mi>
                          <mi mathvariant="italic">i</mi>
                          <mn>0</mn>
                        </msubsup>
                      </mrow></semantics></math></span>.</p>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">An Elastic Network Model (ENM) represents the folded protein as a network of sites connected by springs. They have proved accurate and useful in a variety of applications [<span class="CitationRef"><a href="#CR17">17</a></span>, <span class="CitationRef"><a href="#CR19">19</a></span>]. The potential energy landscape is given by:<div id="Equ1" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mi mathvariant="italic">V</mi>
                          <mfenced open="(" close=")">
                            <mi mathvariant="bold-italic">r</mi>
                          </mfenced>
                          <mo>=</mo>
                          <mfrac>
                            <mn>1</mn>
                            <mn>2</mn>
                          </mfrac>
                          <mstyle displaystyle="true">
                            <munderover>
                              <mi mathsize="big">∑</mi>
                              <mrow>
                                <mi mathvariant="italic">i</mi>
                                <mo>=</mo>
                                <mn>1</mn>
                              </mrow>
                              <mrow>
                                <mi mathvariant="italic">N</mi>
                                <mo>-</mo>
                                <mn>1</mn>
                              </mrow>
                            </munderover>
                          </mstyle>
                          <mstyle displaystyle="true">
                            <munderover>
                              <mi mathsize="big">∑</mi>
                              <mrow>
                                <mi mathvariant="italic">j</mi>
                                <mo>=</mo>
                                <mi mathvariant="italic">i</mi>
                                <mo>+</mo>
                                <mn>1</mn>
                              </mrow>
                              <mi mathvariant="italic">N</mi>
                            </munderover>
                          </mstyle>
                          <msub>
                            <mi mathvariant="italic">k</mi>
                            <mi mathvariant="italic">ij</mi>
                          </msub>
                          <msup>
                            <mfenced open="(" close=")">
                              <mrow>
                                <msub>
                                  <mi mathvariant="italic">d</mi>
                                  <mi mathvariant="italic">ij</mi>
                                </msub>
                                <mo>-</mo>
                                <msubsup>
                                  <mi mathvariant="italic">d</mi>
                                  <mi mathvariant="italic">ij</mi>
                                  <mn>0</mn>
                                </msubsup>
                              </mrow>
                            </mfenced>
                            <mn>2</mn>
                          </msup>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(1)</div></div></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">where <span class="InlineEquation" id="IEq2"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mi mathvariant="italic">d</mi>
                          <mi mathvariant="italic">ij</mi>
                          <mn>0</mn>
                        </msubsup>
                      </mrow></semantics></math></span> and <em xmlns="" class="EmphasisTypeItalic">k</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub> are, respectively, the equilibrium length and force constant of spring <em xmlns="" class="EmphasisTypeItalic">ij</em>. As far as we know, all models proposed so far assume that <span class="InlineEquation" id="IEq3"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msubsup>
                            <mi mathvariant="italic">d</mi>
                            <mi mathvariant="italic">ij</mi>
                            <mn>0</mn>
                          </msubsup>
                          <mo>=</mo>
                          <msub>
                            <mi mathvariant="italic">d</mi>
                            <mi mathvariant="italic">ij</mi>
                          </msub>
                          <mfenced open="(" close=")">
                            <msup>
                              <mi mathvariant="bold-italic">r</mi>
                              <mn>0</mn>
                            </msup>
                          </mfenced>
                          <mo>=</mo>
                          <msubsup>
                            <mi mathvariant="bold-italic">r</mi>
                            <mi mathvariant="italic">j</mi>
                            <mn>0</mn>
                          </msubsup>
                          <mo>-</mo>
                          <msubsup>
                            <mi mathvariant="bold-italic">r</mi>
                            <mi mathvariant="italic">i</mi>
                            <mn>0</mn>
                          </msubsup>
                        </mrow>
                      </mrow></semantics></math></span>, i.e. that at the equilibrium conformation <span xmlns="" class="EmphasisTypeBoldItalic">r</span><sup>0</sup>, all springs are relaxed.</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Fluctuations and flexibility</h3>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">No protein is frozen at its equilibrium conformation. At non-zero absolute temperature, the folded protein fluctuates around &nbsp;<span xmlns="" class="EmphasisTypeBoldItalic">r</span><sup>0</sup> sampling conformational space with equilibrium Boltzmann’s probability density function:<div id="Equ2" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mi mathvariant="italic">ρ</mi>
                          <mfenced open="(" close=")">
                            <mi mathvariant="bold-italic">r</mi>
                          </mfenced>
                          <mo>=</mo>
                          <mfrac>
                            <msup>
                              <mi mathvariant="italic">e</mi>
                              <mrow>
                                <mo>-</mo>
                                <mi mathvariant="italic">βV</mi>
                                <mfenced open="(" close=")">
                                  <mi mathvariant="bold-italic">r</mi>
                                </mfenced>
                              </mrow>
                            </msup>
                            <msub>
                              <mi mathvariant="italic">Z</mi>
                              <mi mathvariant="italic">F</mi>
                            </msub>
                          </mfrac>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(2)</div></div></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">where <span class="InlineEquation" id="IEq4"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mi mathvariant="italic">β</mi>
                          <mo>=</mo>
                          <mfrac bevelled="true">
                            <mn>1</mn>
                            <mrow>
                              <msub>
                                <mi mathvariant="italic">k</mi>
                                <mi mathvariant="italic">B</mi>
                              </msub>
                              <mi mathvariant="italic">T</mi>
                            </mrow>
                          </mfrac>
                        </mrow>
                      </mrow></semantics></math></span>, with <em xmlns="" class="EmphasisTypeItalic">T</em> the absolute temperature and <em xmlns="" class="EmphasisTypeItalic">k</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">B</em>
                  </sub> Boltzmann’s constant. The denominator of Eq. (<span class="InternalRef"><a href="#Equ2">2</a></span>) is the partition function of the folded protein:<div id="Equ3" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msub>
                            <mi mathvariant="italic">Z</mi>
                            <mi mathvariant="italic">F</mi>
                          </msub>
                          <mo>=</mo>
                          <mstyle displaystyle="true">
                            <mo>∫</mo>
                          </mstyle>
                          <msup>
                            <mi mathvariant="italic">e</mi>
                            <mrow>
                              <mo>-</mo>
                              <mi mathvariant="italic">βV</mi>
                              <mfenced open="(" close=")">
                                <mi mathvariant="bold-italic">r</mi>
                              </mfenced>
                            </mrow>
                          </msup>
                          <mi mathvariant="italic">dτ</mi>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(3)</div></div></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">where ∫ … <em xmlns="" class="EmphasisTypeItalic">dτ</em> stands for integration over the whole of conformational space.</p>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">The dynamical flexibility (mobility) of a site is ordinarily quantified using its Mean Square Fluctuation:<div id="Equ4" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mi mathvariant="italic">MS</mi>
                          <msub>
                            <mi mathvariant="italic">F</mi>
                            <mi mathvariant="italic">i</mi>
                          </msub>
                          <mo>≡</mo>
                          <mfenced open="〈" close="〉">
                            <msup>
                              <mfenced open="║" close="║">
                                <mrow>
                                  <msub>
                                    <mi mathvariant="bold-italic">r</mi>
                                    <mi mathvariant="italic">i</mi>
                                  </msub>
                                  <mo>-</mo>
                                  <msubsup>
                                    <mi mathvariant="bold-italic">r</mi>
                                    <mi mathvariant="italic">i</mi>
                                    <mn>0</mn>
                                  </msubsup>
                                </mrow>
                              </mfenced>
                              <mn>2</mn>
                            </msup>
                          </mfenced>
                          <mo>=</mo>
                          <mstyle displaystyle="true">
                            <mo>∫</mo>
                          </mstyle>
                          <msup>
                            <mfenced open="║" close="║">
                              <mrow>
                                <msub>
                                  <mi mathvariant="bold-italic">r</mi>
                                  <mi mathvariant="italic">i</mi>
                                </msub>
                                <mo>-</mo>
                                <msubsup>
                                  <mi mathvariant="bold-italic">r</mi>
                                  <mi mathvariant="italic">i</mi>
                                  <mn>0</mn>
                                </msubsup>
                              </mrow>
                            </mfenced>
                            <mn>2</mn>
                          </msup>
                          <mi mathvariant="italic">ρ</mi>
                          <mfenced open="(" close=")">
                            <mi mathvariant="bold-italic">r</mi>
                          </mfenced>
                          <mi mathvariant="italic">dτ</mi>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(4)</div></div></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">To calculate <em xmlns="" class="EmphasisTypeItalic">MSF</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub> using Eq. (<span class="InternalRef"><a href="#Equ4">4</a></span>), the potential energy function Eq. (<span class="InternalRef"><a href="#Equ1">1</a></span>) is approximated using a second-order Taylor expansion around its equilibrium conformation. First, the Hessian matrix <span xmlns="" class="EmphasisTypeBoldItalic">H</span> of second derivatives of the potential Eq. (<span class="InternalRef"><a href="#Equ1">1</a></span>) with respect to the atoms’ Cartesian coordinates is calculated. Then, <span xmlns="" class="EmphasisTypeBoldItalic">H</span> is inverted to obtain the 3<em xmlns="" class="EmphasisTypeItalic">N</em> × 3<em xmlns="" class="EmphasisTypeItalic">N</em> variance-covariance matrix <span xmlns="" class="EmphasisTypeBoldItalic">C</span>, which is composed of a 3 × 3 <span xmlns="" class="EmphasisTypeBoldItalic">C</span><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub> block for each pair of sites. Finally, a site’s MSF is given by [<span class="CitationRef"><a href="#CR20">20</a></span>]:<div id="Equ5" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mi mathvariant="italic">MS</mi>
                          <msub>
                            <mi mathvariant="italic">F</mi>
                            <mi mathvariant="italic">i</mi>
                          </msub>
                          <mo>=</mo>
                          <mi mathvariant="italic">Tr</mi>
                          <mfenced open="(" close=")">
                            <msub>
                              <mi mathvariant="bold-italic">C</mi>
                              <mi mathvariant="italic">ii</mi>
                            </msub>
                          </mfenced>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(5)</div></div></div>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">An empirical flexibility model</h3>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">Several studies have investigated the correlation between site-specific rates of evolution or other sequence-variability measures and the corresponding flexibility. Since such studies use Pearson’s correlation coefficients as measure of association, the underlying assumption is that there is a linear relationship between rate of evolution and flexibility. To make such assumption explicit, here we postulate the following Flexibility Model:<div id="Equ6" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msup>
                            <mover accent="true">
                              <msub>
                                <mi mathvariant="italic">ω</mi>
                                <mi mathvariant="italic">i</mi>
                              </msub>
                              <mo stretchy="true">˜</mo>
                            </mover>
                            <mi mathvariant="italic">FLEX</mi>
                          </msup>
                          <mo>=</mo>
                          <msubsup>
                            <mi mathvariant="italic">a</mi>
                            <mi mathvariant="italic">P</mi>
                            <mi mathvariant="italic">FLEX</mi>
                          </msubsup>
                          <mo>+</mo>
                          <msubsup>
                            <mi mathvariant="italic">b</mi>
                            <mi mathvariant="italic">P</mi>
                            <mi mathvariant="italic">FLEX</mi>
                          </msubsup>
                          <mover accent="true">
                            <mrow>
                              <mi mathvariant="italic">MS</mi>
                              <msub>
                                <mi mathvariant="italic">F</mi>
                                <mi mathvariant="italic">i</mi>
                              </msub>
                            </mrow>
                            <mo stretchy="true">˜</mo>
                          </mover>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(6)</div></div></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">where <span class="InlineEquation" id="IEq5"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mover accent="true">
                            <msub>
                              <mi mathvariant="italic">ω</mi>
                              <mi mathvariant="italic">i</mi>
                            </msub>
                            <mo stretchy="true">˜</mo>
                          </mover>
                          <mspace width="0.25em"></mspace>
                        </mrow>
                      </mrow></semantics></math></span> is the relative rate of substitution of the <em xmlns="" class="EmphasisTypeItalic">i</em>th site. In general, for site-specific scalar properties we will use relative values obtained by z-score normalization. For any given site-specific property <em xmlns="" class="EmphasisTypeItalic">x</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>, we the z-score normalized values are <span class="InlineEquation" id="IEq6"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mover accent="true">
                            <msub>
                              <mi mathvariant="italic">x</mi>
                              <mi mathvariant="italic">i</mi>
                            </msub>
                            <mo stretchy="true">˜</mo>
                          </mover>
                          <mo>=</mo>
                          <mfrac bevelled="true">
                            <mfenced open="(" close=")">
                              <mrow>
                                <msub>
                                  <mi mathvariant="italic">x</mi>
                                  <mi mathvariant="italic">i</mi>
                                </msub>
                                <mo>-</mo>
                                <mfenced open="〈" close="〉">
                                  <mi mathvariant="italic">x</mi>
                                </mfenced>
                              </mrow>
                            </mfenced>
                            <msqrt>
                              <mrow>
                                <mfenced open="〈" close="〉">
                                  <msup>
                                    <mi mathvariant="italic">x</mi>
                                    <mn>2</mn>
                                  </msup>
                                </mfenced>
                                <mo>-</mo>
                                <mfenced open="〈" close="〉">
                                  <msup>
                                    <mi mathvariant="italic">x</mi>
                                    <mn>2</mn>
                                  </msup>
                                </mfenced>
                              </mrow>
                            </msqrt>
                          </mfrac>
                        </mrow>
                      </mrow></semantics></math></span>, where the averages are calculated over all sites of the same protein. The subscript <em xmlns="" class="EmphasisTypeItalic">P</em> is used to note that <em xmlns="" class="EmphasisTypeItalic">a priori</em> the coefficients may depend on the protein considered. We emphasize that the Flexibility Model is empirical: rather than derived from first principles, it is postulated, based on the intuitive notion that flexible sites should accommodate mutations more easily.</p>
              </section><section id="Sec6" class="Section2 RenderAsSection2">
                <h3 class="Heading">A mechanistic stress model</h3>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">We introduce here a mechanistic model that includes explicitly the effects of mutations and natural selection. We consider mutations as random perturbations of the wild-type ENM potential [<span class="CitationRef"><a href="#CR21">21</a></span>–<span class="CitationRef"><a href="#CR23">23</a></span>]. A random mutation at site <em xmlns="" class="EmphasisTypeItalic">i</em> results in a mutant whose potential <em xmlns="" class="EmphasisTypeItalic">V</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">mut</em>
                  </sub> is obtained from Eq. (<span class="InternalRef"><a href="#Equ1">1</a></span>) by adding perturbations to the equilibrium length of each of its springs: <span class="InlineEquation" id="IEq7"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msubsup>
                            <mi mathvariant="italic">d</mi>
                            <mi mathvariant="italic">ij</mi>
                            <mn>0</mn>
                          </msubsup>
                          <mo>→</mo>
                          <msubsup>
                            <mi mathvariant="italic">d</mi>
                            <mi mathvariant="italic">ij</mi>
                            <mn>0</mn>
                          </msubsup>
                          <mo>+</mo>
                          <msub>
                            <mi mathvariant="italic">δ</mi>
                            <mi mathvariant="italic">ij</mi>
                          </msub>
                          <mo>.</mo>
                        </mrow>
                      </mrow></semantics></math></span> We further assume that the springs are independently perturbed and that perturbations are spring-independent, randomly drawn from a distribution with zero mean and constant variance <em xmlns="" class="EmphasisTypeItalic">α</em><sup>2</sup>:<div id="Equ7" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mfenced open="〈" close="〉">
                            <msub>
                              <mi mathvariant="italic">δ</mi>
                              <mi mathvariant="italic">ij</mi>
                            </msub>
                          </mfenced>
                          <mo>=</mo>
                          <mn>0</mn>
                          <mo>;</mo>
                          <mfenced open="〈" close="〉">
                            <msubsup>
                              <mi mathvariant="italic">δ</mi>
                              <mi mathvariant="italic">ij</mi>
                              <mn>2</mn>
                            </msubsup>
                          </mfenced>
                          <mo>=</mo>
                          <msup>
                            <mi mathvariant="italic">α</mi>
                            <mn>2</mn>
                          </msup>
                          <mo>.</mo>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(7)</div></div></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">As we mentioned above, when the wild type is at its equilibrium conformation <span class="InlineEquation" id="IEq8"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mi mathvariant="bold-italic">r</mi>
                          <mi mathvariant="italic">wt</mi>
                          <mn>0</mn>
                        </msubsup>
                      </mrow></semantics></math></span>, all springs are relaxed by construction. In contrast, when the mutant is at <span class="InlineEquation" id="IEq9"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mi mathvariant="bold-italic">r</mi>
                          <mi mathvariant="italic">wt</mi>
                          <mn>0</mn>
                        </msubsup>
                      </mrow></semantics></math></span>, the mutated site’s springs will be stressed (stretched or compressed). For further reference, we define the Mean Local mutational Stress (MLmS) as follows:<div id="Equ8" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mi mathvariant="italic">MLm</mi>
                          <msub>
                            <mi mathvariant="italic">S</mi>
                            <mi mathvariant="italic">i</mi>
                          </msub>
                          <mo>≡</mo>
                          <msub>
                            <mfenced open="〈" close="〉">
                              <mrow>
                                <msub>
                                  <mi mathvariant="italic">V</mi>
                                  <mi mathvariant="italic">mut</mi>
                                </msub>
                                <mfenced open="(" close=")">
                                  <msubsup>
                                    <mi mathvariant="bold-italic">r</mi>
                                    <mi mathvariant="italic">wt</mi>
                                    <mn>0</mn>
                                  </msubsup>
                                </mfenced>
                                <mo>-</mo>
                                <msub>
                                  <mi mathvariant="italic">V</mi>
                                  <mi mathvariant="italic">wt</mi>
                                </msub>
                                <mfenced open="(" close=")">
                                  <msubsup>
                                    <mi mathvariant="bold-italic">r</mi>
                                    <mi mathvariant="italic">wt</mi>
                                    <mn>0</mn>
                                  </msubsup>
                                </mfenced>
                              </mrow>
                            </mfenced>
                            <mrow>
                              <mi mathvariant="italic">mut</mi>
                              <mo>@</mo>
                              <mi mathvariant="italic">i</mi>
                            </mrow>
                          </msub>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(8)</div></div></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">where 〈 … 〉<sub><em xmlns="" class="EmphasisTypeItalic">mut</em> @ <em xmlns="" class="EmphasisTypeItalic">i</em></sub> stands for averaging over random mutations at the <em xmlns="" class="EmphasisTypeItalic">i</em>th site.</p>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">To complete the model, we derive a simple selection function. First, we assume that there is a single specific active conformation <span xmlns="" class="EmphasisTypeBoldItalic">r</span><sub>
                    <em xmlns="" class="EmphasisTypeItalic">active</em>
                  </sub>. Next, we acknowledge fluctuations and assume that the protein’s activity (either the wild-type’s or a mutant’s) is proportional to the concentration of the active conformation <span xmlns="" class="EmphasisTypeBoldItalic">r</span><sub>
                    <em xmlns="" class="EmphasisTypeItalic">active</em>
                  </sub>. Finally, we assume that <span class="InlineEquation" id="IEq10"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msub>
                            <mi mathvariant="bold-italic">r</mi>
                            <mi mathvariant="italic">active</mi>
                          </msub>
                          <mo>=</mo>
                          <msubsup>
                            <mi mathvariant="bold-italic">r</mi>
                            <mi mathvariant="italic">wt</mi>
                            <mn>0</mn>
                          </msubsup>
                        </mrow>
                      </mrow></semantics></math></span> and, accordingly, we model the acceptance probability of a mutant as:<div id="Equ9" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msup>
                            <mi mathvariant="italic">p</mi>
                            <mi mathvariant="italic">accept</mi>
                          </msup>
                          <mo>≡</mo>
                          <mfrac>
                            <mrow>
                              <msubsup>
                                <mi mathvariant="italic">C</mi>
                                <mi mathvariant="italic">mut</mi>
                                <mi mathvariant="italic">F</mi>
                              </msubsup>
                              <msub>
                                <mi mathvariant="italic">ρ</mi>
                                <mi mathvariant="italic">mut</mi>
                              </msub>
                              <mfenced open="(" close=")">
                                <msubsup>
                                  <mi mathvariant="bold-italic">r</mi>
                                  <mi mathvariant="italic">wt</mi>
                                  <mn>0</mn>
                                </msubsup>
                              </mfenced>
                            </mrow>
                            <mrow>
                              <msubsup>
                                <mi mathvariant="italic">C</mi>
                                <mi mathvariant="italic">wt</mi>
                                <mi mathvariant="italic">F</mi>
                              </msubsup>
                              <msub>
                                <mi mathvariant="italic">ρ</mi>
                                <mi mathvariant="italic">wt</mi>
                              </msub>
                              <mfenced open="(" close=")">
                                <msubsup>
                                  <mi mathvariant="bold-italic">r</mi>
                                  <mi mathvariant="italic">wt</mi>
                                  <mn>0</mn>
                                </msubsup>
                              </mfenced>
                            </mrow>
                          </mfrac>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(9)</div></div></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">Where <span class="InlineEquation" id="IEq11"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mi mathvariant="italic">C</mi>
                          <mi mathvariant="italic">mut</mi>
                          <mi mathvariant="italic">F</mi>
                        </msubsup>
                      </mrow></semantics></math></span> and <span class="InlineEquation" id="IEq12"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mi mathvariant="italic">C</mi>
                          <mi mathvariant="italic">wt</mi>
                          <mi mathvariant="italic">F</mi>
                        </msubsup>
                      </mrow></semantics></math></span> are the concentrations of folded protein for the mutant and wild type, respectively. From statistical mechanics, the Folded-Unfolded equilibrium constants for the wild-type and mutant proteins are, respectively, <span class="InlineEquation" id="IEq13"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mfrac bevelled="true">
                            <msubsup>
                              <mi mathvariant="italic">C</mi>
                              <mi mathvariant="italic">wt</mi>
                              <mi mathvariant="italic">F</mi>
                            </msubsup>
                            <msubsup>
                              <mi mathvariant="italic">C</mi>
                              <mi mathvariant="italic">wt</mi>
                              <mi mathvariant="italic">U</mi>
                            </msubsup>
                          </mfrac>
                          <mo>=</mo>
                          <mfrac bevelled="true">
                            <msubsup>
                              <mi mathvariant="italic">Z</mi>
                              <mi mathvariant="italic">wt</mi>
                              <mi mathvariant="italic">F</mi>
                            </msubsup>
                            <msubsup>
                              <mi mathvariant="italic">Z</mi>
                              <mi mathvariant="italic">wt</mi>
                              <mi mathvariant="italic">U</mi>
                            </msubsup>
                          </mfrac>
                        </mrow>
                      </mrow></semantics></math></span> and <span class="InlineEquation" id="IEq14"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mfrac bevelled="true">
                            <msubsup>
                              <mi mathvariant="italic">C</mi>
                              <mi mathvariant="italic">mut</mi>
                              <mi mathvariant="italic">F</mi>
                            </msubsup>
                            <msubsup>
                              <mi mathvariant="italic">C</mi>
                              <mi mathvariant="italic">mut</mi>
                              <mi mathvariant="italic">U</mi>
                            </msubsup>
                          </mfrac>
                          <mo>=</mo>
                          <mfrac bevelled="true">
                            <msubsup>
                              <mi mathvariant="italic">Z</mi>
                              <mi mathvariant="italic">mut</mi>
                              <mi mathvariant="italic">F</mi>
                            </msubsup>
                            <msubsup>
                              <mi mathvariant="italic">Z</mi>
                              <mi mathvariant="italic">mut</mi>
                              <mi mathvariant="italic">U</mi>
                            </msubsup>
                          </mfrac>
                        </mrow>
                      </mrow></semantics></math></span>. We further assume that the partition function and concentration of unfolded protein is the same for the mutant and wild type. Therefore <span class="InlineEquation" id="IEq15"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mfrac bevelled="true">
                            <msubsup>
                              <mi mathvariant="italic">C</mi>
                              <mi mathvariant="italic">mut</mi>
                              <mi mathvariant="italic">F</mi>
                            </msubsup>
                            <msubsup>
                              <mi mathvariant="italic">C</mi>
                              <mi mathvariant="italic">wt</mi>
                              <mi mathvariant="italic">F</mi>
                            </msubsup>
                          </mfrac>
                          <mo>=</mo>
                          <mfrac bevelled="true">
                            <msub>
                              <mi mathvariant="italic">Z</mi>
                              <mi mathvariant="italic">mut</mi>
                            </msub>
                            <msub>
                              <mi mathvariant="italic">Z</mi>
                              <mi mathvariant="italic">wt</mi>
                            </msub>
                          </mfrac>
                        </mrow>
                      </mrow></semantics></math></span>. Replacing this relationship and Eq. (<span class="InternalRef"><a href="#Equ2">2</a></span>) into Eq. (<span class="InternalRef"><a href="#Equ9">9</a></span>) we find:<div id="Equ10" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msup>
                            <mi mathvariant="italic">p</mi>
                            <mi mathvariant="italic">accept</mi>
                          </msup>
                          <mo>=</mo>
                          <msup>
                            <mi mathvariant="italic">e</mi>
                            <mrow>
                              <mo>-</mo>
                              <mi mathvariant="italic">β</mi>
                              <mspace width="0.25em"></mspace>
                              <mfenced open="[" close="]">
                                <mrow>
                                  <msub>
                                    <mi mathvariant="italic">V</mi>
                                    <mi mathvariant="italic">mut</mi>
                                  </msub>
                                  <mfenced open="(" close=")">
                                    <msubsup>
                                      <mi mathvariant="bold-italic">r</mi>
                                      <mi mathvariant="italic">wt</mi>
                                      <mn>0</mn>
                                    </msubsup>
                                  </mfenced>
                                  <mo>-</mo>
                                  <msub>
                                    <mi mathvariant="italic">V</mi>
                                    <mi mathvariant="italic">wt</mi>
                                  </msub>
                                  <mfenced open="(" close=")">
                                    <msubsup>
                                      <mi mathvariant="bold-italic">r</mi>
                                      <mi mathvariant="italic">wt</mi>
                                      <mn>0</mn>
                                    </msubsup>
                                  </mfenced>
                                </mrow>
                              </mfenced>
                            </mrow>
                          </msup>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(10)</div></div></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">Finally, averaging over random mutations at site <em xmlns="" class="EmphasisTypeItalic">i</em> and using Eq. (<span class="InternalRef"><a href="#Equ8">8</a></span>) we obtain the <em xmlns="" class="EmphasisTypeItalic">acceptance rate</em>:<div id="Equ11" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msub>
                            <mi mathvariant="italic">ω</mi>
                            <mi mathvariant="italic">i</mi>
                          </msub>
                          <mo>≡</mo>
                          <msub>
                            <mfenced open="〈" close="〉">
                              <msubsup>
                                <mi mathvariant="italic">p</mi>
                                <mi mathvariant="italic">i</mi>
                                <mi mathvariant="italic">accept</mi>
                              </msubsup>
                            </mfenced>
                            <mrow>
                              <mi mathvariant="italic">mut</mi>
                              <mo>@</mo>
                              <mi mathvariant="italic">i</mi>
                            </mrow>
                          </msub>
                          <mo>=</mo>
                          <msub>
                            <mfenced open="〈" close="〉">
                              <msup>
                                <mi mathvariant="italic">e</mi>
                                <mrow>
                                  <mo>-</mo>
                                  <mi mathvariant="italic">β</mi>
                                  <mspace width="0.25em"></mspace>
                                  <mfenced open="[" close="]">
                                    <mrow>
                                      <msub>
                                        <mi mathvariant="italic">V</mi>
                                        <mi mathvariant="italic">mut</mi>
                                      </msub>
                                      <mfenced open="(" close=")">
                                        <msubsup>
                                          <mi mathvariant="bold-italic">r</mi>
                                          <mi mathvariant="italic">wt</mi>
                                          <mn>0</mn>
                                        </msubsup>
                                      </mfenced>
                                      <mo>-</mo>
                                      <msub>
                                        <mi mathvariant="italic">V</mi>
                                        <mi mathvariant="italic">wt</mi>
                                      </msub>
                                      <mfenced open="(" close=")">
                                        <msubsup>
                                          <mi mathvariant="bold-italic">r</mi>
                                          <mi mathvariant="italic">wt</mi>
                                          <mn>0</mn>
                                        </msubsup>
                                      </mfenced>
                                    </mrow>
                                  </mfenced>
                                </mrow>
                              </msup>
                            </mfenced>
                            <mrow>
                              <mi mathvariant="italic">mut</mi>
                              <mo>@</mo>
                              <mi mathvariant="italic">i</mi>
                            </mrow>
                          </msub>
                          <mo>≅</mo>
                          <mn>1</mn>
                          <mo>-</mo>
                          <mi mathvariant="italic">β</mi>
                          <mspace width="0.25em"></mspace>
                          <mi mathvariant="italic">MLm</mi>
                          <msub>
                            <mi mathvariant="italic">S</mi>
                            <mi mathvariant="italic">i</mi>
                          </msub>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(11)</div></div></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">Where <em xmlns="" class="EmphasisTypeItalic">β</em> may be thought of as representing not just temperature but also selection pressure, and we have assumed that <em xmlns="" class="EmphasisTypeItalic">β</em>Δ<em xmlns="" class="EmphasisTypeItalic">V</em> &lt;&lt; 1 (mild selection) to approximate the exponential to first order. To finish, we z-normalize the variables of Eq. (<span class="InternalRef"><a href="#Equ11">11</a></span>) to get the relative rates of evolution:<div id="Equ12" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msup>
                            <mover accent="true">
                              <msub>
                                <mi mathvariant="italic">ω</mi>
                                <mi mathvariant="italic">i</mi>
                              </msub>
                              <mo stretchy="true">˜</mo>
                            </mover>
                            <mi mathvariant="italic">STRESS</mi>
                          </msup>
                          <mo>=</mo>
                          <msubsup>
                            <mi mathvariant="italic">a</mi>
                            <mi mathvariant="italic">P</mi>
                            <mi mathvariant="italic">STRESS</mi>
                          </msubsup>
                          <mo>+</mo>
                          <msubsup>
                            <mi mathvariant="italic">b</mi>
                            <mi mathvariant="italic">P</mi>
                            <mi mathvariant="italic">STRESS</mi>
                          </msubsup>
                          <mover accent="true">
                            <mrow>
                              <mi mathvariant="italic">MLm</mi>
                              <msub>
                                <mi mathvariant="italic">S</mi>
                                <mi mathvariant="italic">i</mi>
                              </msub>
                            </mrow>
                            <mo stretchy="true">˜</mo>
                          </mover>
                          <mo>.</mo>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(12)</div></div></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">This equation specifies the stress model.</p>
              </section><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">Relationship of flexibility and stress with packing density</h3>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">The purpose of this work is to investigate why LPD correlates with rate of evolution at site level. The previous models relate rates of evolution with MSF (Eq.&nbsp;<span class="InternalRef"><a href="#Equ6">6</a></span>) and MLmS (Eq.&nbsp;<span class="InternalRef"><a href="#Equ12">12</a></span>). Here we derive the relationship between these properties and LPD measures.</p>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">First, we relate flexibility and stress with the potential energy parameters of Eq. (<span class="InternalRef"><a href="#Equ1">1</a></span>). Let us define:<div id="Equ13" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msub>
                            <mi mathvariant="italic">k</mi>
                            <mi mathvariant="italic">i</mi>
                          </msub>
                          <mo>≡</mo>
                          <mstyle displaystyle="true">
                            <munder>
                              <mi mathsize="big">∑</mi>
                              <mrow>
                                <mi mathvariant="italic">j</mi>
                                <mo>≠</mo>
                                <mi mathvariant="italic">i</mi>
                              </mrow>
                            </munder>
                          </mstyle>
                          <msub>
                            <mi mathvariant="italic">k</mi>
                            <mi mathvariant="italic">ij</mi>
                          </msub>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(13)</div></div></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">Regarding flexibility, replacing Eqs. (<span class="InternalRef"><a href="#Equ1">1</a></span>), (2), and (3) into Eq. (<span class="InternalRef"><a href="#Equ4">4</a></span>), following [<span class="CitationRef"><a href="#CR12">12</a></span>], and using Eq. (<span class="InternalRef"><a href="#Equ13">13</a></span>), it can be found that:<div id="Equ14" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mi mathvariant="italic">MS</mi>
                          <msub>
                            <mi mathvariant="italic">F</mi>
                            <mi mathvariant="italic">i</mi>
                          </msub>
                          <mo>≅</mo>
                          <mfrac>
                            <mn>3</mn>
                            <mrow>
                              <mn>2</mn>
                              <mi mathvariant="italic">β</mi>
                              <msub>
                                <mi mathvariant="italic">k</mi>
                                <mi mathvariant="italic">i</mi>
                              </msub>
                            </mrow>
                          </mfrac>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(14)</div></div></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">Regarding stress, from Eqs. (<span class="InternalRef"><a href="#Equ1">1</a></span>), (7), and (8), after some algebra, we get:<div id="Equ15" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mi mathvariant="italic">MLm</mi>
                          <msub>
                            <mi mathvariant="italic">S</mi>
                            <mi mathvariant="italic">i</mi>
                          </msub>
                          <mo>=</mo>
                          <mfrac>
                            <mn>1</mn>
                            <mn>2</mn>
                          </mfrac>
                          <msup>
                            <mi mathvariant="italic">α</mi>
                            <mn>2</mn>
                          </msup>
                          <msub>
                            <mi mathvariant="italic">k</mi>
                            <mi mathvariant="italic">i</mi>
                          </msub>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(15)</div></div></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Note that Eq. (<span class="InternalRef"><a href="#Equ14">14</a></span>) is an approximation while Eq. (<span class="InternalRef"><a href="#Equ15">15</a></span>) is an identity.</p>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">Second, to relate the previous models to LPD we need to specify the ENMs spring constants. A variety of ENMs have been developed (see [<span class="CitationRef"><a href="#CR24">24</a></span>] for a recent comparison). Here, we consider two models. First, the “parameter-free Anisotropic Network Model” (pfANM) [<span class="CitationRef"><a href="#CR25">25</a></span>], which uses:<div id="Equ16" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msub>
                            <mi mathvariant="italic">k</mi>
                            <mi mathvariant="italic">ij</mi>
                          </msub>
                          <mo>=</mo>
                          <mfrac>
                            <mn>1</mn>
                            <msup>
                              <mfenced open="(" close=")">
                                <msubsup>
                                  <mi mathvariant="italic">d</mi>
                                  <mi mathvariant="italic">ij</mi>
                                  <mn>0</mn>
                                </msubsup>
                              </mfenced>
                              <mn>2</mn>
                            </msup>
                          </mfrac>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(16)</div></div></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">Second, the “Anisotropic Network Model” (ANM) [<span class="CitationRef"><a href="#CR20">20</a></span>], for which:<div id="Equ17" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msub>
                            <mi mathvariant="italic">k</mi>
                            <mi mathvariant="italic">ij</mi>
                          </msub>
                          <mo>=</mo>
                          <mfenced open="{" close="">
                            <mtable columnalign="center">
                              <mtr columnalign="center">
                                <mtd columnalign="center">
                                  <mn>1</mn>
                                </mtd>
                                <mtd columnalign="center">
                                  <msubsup>
                                    <mi mathvariant="italic">d</mi>
                                    <mi mathvariant="italic">ij</mi>
                                    <mn>0</mn>
                                  </msubsup>
                                  <mo>≤</mo>
                                  <msub>
                                    <mi mathvariant="italic">R</mi>
                                    <mi mathvariant="italic">cut</mi>
                                  </msub>
                                </mtd>
                              </mtr>
                              <mtr columnalign="center">
                                <mtd columnalign="center">
                                  <mn>0</mn>
                                </mtd>
                                <mtd columnalign="center">
                                  <msubsup>
                                    <mi mathvariant="italic">d</mi>
                                    <mi mathvariant="italic">ij</mi>
                                    <mn>0</mn>
                                  </msubsup>
                                  <mo>&gt;</mo>
                                  <msub>
                                    <mi mathvariant="italic">R</mi>
                                    <mi mathvariant="italic">cut</mi>
                                  </msub>
                                </mtd>
                              </mtr>
                            </mtable>
                          </mfenced>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(17)</div></div></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">where <em xmlns="" class="EmphasisTypeItalic">R</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">cut</em>
                  </sub> is typically between 10&nbsp;Å and 18&nbsp;Å.</p>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">From Eqs. (<span class="InternalRef"><a href="#Equ13">13</a></span>), (16), and (17) and z-normalizing we find:<div id="Equ18" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mover accent="true">
                            <msub>
                              <mi mathvariant="italic">k</mi>
                              <mi mathvariant="italic">i</mi>
                            </msub>
                            <mo stretchy="true">˜</mo>
                          </mover>
                          <mo>=</mo>
                          <mover accent="true">
                            <mrow>
                              <mi mathvariant="italic">LP</mi>
                              <msub>
                                <mi mathvariant="italic">D</mi>
                                <mi mathvariant="italic">i</mi>
                              </msub>
                            </mrow>
                            <mo stretchy="true">˜</mo>
                          </mover>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(18)</div></div></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">where for the pfANM, LPD is the Weighted Contact Number (WCN) of [<span class="CitationRef"><a href="#CR26">26</a></span>], and for the ANM, it is the Contact Number (CN): the number of sites closer than <em xmlns="" class="EmphasisTypeItalic">R</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">cut</em>
                  </sub>. Finally, from Eqs. (<span class="InternalRef"><a href="#Equ14">14</a></span>) and (<span class="InternalRef"><a href="#Equ18">18</a></span>) it follows:<div id="Equ19" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mover accent="true">
                            <mrow>
                              <mi mathvariant="italic">MS</mi>
                              <msub>
                                <mi mathvariant="italic">F</mi>
                                <mi mathvariant="italic">i</mi>
                              </msub>
                            </mrow>
                            <mo stretchy="true">˜</mo>
                          </mover>
                          <mo>≅</mo>
                          <msup>
                            <mover accent="true">
                              <msub>
                                <mi mathvariant="italic">k</mi>
                                <mi mathvariant="italic">i</mi>
                              </msub>
                              <mo stretchy="true">˜</mo>
                            </mover>
                            <mrow>
                              <mo>-</mo>
                              <mn>1</mn>
                            </mrow>
                          </msup>
                          <mo>=</mo>
                          <msup>
                            <mover accent="true">
                              <mrow>
                                <mi mathvariant="italic">LP</mi>
                                <msub>
                                  <mi mathvariant="italic">D</mi>
                                  <mi mathvariant="italic">i</mi>
                                </msub>
                              </mrow>
                              <mo stretchy="true">˜</mo>
                            </mover>
                            <mrow>
                              <mo>-</mo>
                              <mn>1</mn>
                            </mrow>
                          </msup>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(19)</div></div></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">Similarly, from Eqs. (<span class="InternalRef"><a href="#Equ15">15</a></span>) and (<span class="InternalRef"><a href="#Equ18">18</a></span>) we get:<div id="Equ20" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mover accent="true">
                            <mrow>
                              <mi mathvariant="italic">MLm</mi>
                              <msub>
                                <mi mathvariant="italic">S</mi>
                                <mi mathvariant="italic">i</mi>
                              </msub>
                            </mrow>
                            <mo stretchy="true">˜</mo>
                          </mover>
                          <mo>=</mo>
                          <mover accent="true">
                            <msub>
                              <mi mathvariant="italic">k</mi>
                              <mi mathvariant="italic">i</mi>
                            </msub>
                            <mo stretchy="true">˜</mo>
                          </mover>
                          <mo>=</mo>
                          <mover accent="true">
                            <mrow>
                              <mi mathvariant="italic">LP</mi>
                              <msub>
                                <mi mathvariant="italic">D</mi>
                                <mi mathvariant="italic">i</mi>
                              </msub>
                            </mrow>
                            <mo stretchy="true">˜</mo>
                          </mover>
                        </mrow>
                      </mrow></semantics></math></div> <div class="EquationNumber">(20)</div></div></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Note that while MSF is approximately equal to 1/LPD, MLmS is exactly equal to LPD (for relative z-normalized values).</p>
              </section><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">Calculation details</h3>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">We used the dataset of 213 monomeric enzymes of Yeh et al. [<span class="CitationRef"><a href="#CR11">11</a></span>]. The dataset includes proteins of diverse sizes, functional, and structural classes (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Table S1).</p>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">We used the evolutionary rates of [<span class="CitationRef"><a href="#CR11">11</a></span>]. They were inferred from multiple alignments of homologous sequences using Rate4Site, which builds the phylogenetic tree using a neighbour-joining algorithm and estimates rates with an empirical Bayesian approach and the JTT model of sequence evolution [<span class="CitationRef"><a href="#CR27">27</a></span>, <span class="CitationRef"><a href="#CR28">28</a></span>]. To keep in mind that we are not dealing with the (unknown) “true rates”, but with Rate4Site-inferred rates, we use the notation <span class="InlineEquation" id="IEq16"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mover accent="true">
                            <mi mathvariant="italic">ω</mi>
                            <mo stretchy="true">˜</mo>
                          </mover>
                          <mi mathvariant="italic">i</mi>
                          <mrow>
                            <mi mathvariant="italic">R</mi>
                            <mn>4</mn>
                            <mi mathvariant="italic">S</mi>
                          </mrow>
                        </msubsup>
                      </mrow></semantics></math></span>.</p>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">From the pdb equilibrium structure of each protein we calculated the spring constants of pfANM (Eq.&nbsp;<span class="InternalRef"><a href="#Equ16">16</a></span>) and ANM (Eq.&nbsp;<span class="InternalRef"><a href="#Equ17">17</a></span>), for which we used a cut-off distance of 13&nbsp;Å [<span class="CitationRef"><a href="#CR11">11</a></span>]. Given a protein and ENM model, we calculated the Hessian matrix, inverted it to obtain the variance-covariance matrix, and calculated the site-specific flexibility values <span class="InlineEquation" id="IEq17"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mover accent="true">
                          <mrow>
                            <mi mathvariant="italic">MS</mi>
                            <msub>
                              <mi mathvariant="italic">F</mi>
                              <mi mathvariant="italic">i</mi>
                            </msub>
                          </mrow>
                          <mo stretchy="true">˜</mo>
                        </mover>
                      </mrow></semantics></math></span> using Eq. (<span class="InternalRef"><a href="#Equ5">5</a></span>) and z-normalizing. Regarding stress, we obtained the relative site-specific values <span class="InlineEquation" id="IEq18"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mover accent="true">
                          <mrow>
                            <mi mathvariant="italic">MLm</mi>
                            <msub>
                              <mi mathvariant="italic">S</mi>
                              <mi mathvariant="italic">i</mi>
                            </msub>
                          </mrow>
                          <mo stretchy="true">˜</mo>
                        </mover>
                      </mrow></semantics></math></span> using Eq. (<span class="InternalRef"><a href="#Equ15">15</a></span>) and z-normalizing.</p>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Since we always use z-normalized relative values, for the sake of notational simplicity, we shall use <em xmlns="" class="EmphasisTypeItalic">ω</em><sup><em xmlns="" class="EmphasisTypeItalic">R</em>4<em xmlns="" class="EmphasisTypeItalic">S</em></sup>, <em xmlns="" class="EmphasisTypeItalic">MSF</em>, and <em xmlns="" class="EmphasisTypeItalic">MLmS</em> to refer to z-normalized values from now on.</p>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">We performed two analyses. In a protein-by-protein analysis, we performed linear fits of <em xmlns="" class="EmphasisTypeItalic">ω</em><sup><em xmlns="" class="EmphasisTypeItalic">R</em>4<em xmlns="" class="EmphasisTypeItalic">S</em></sup> with either <em xmlns="" class="EmphasisTypeItalic">MSF</em> (Flexibility Model) or <em xmlns="" class="EmphasisTypeItalic">MLmS</em> (Stress Model) using the lm() function of the base package of R for each protein. In a global analysis we pooled together all sites of all proteins and performed similar global fits.</p>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">To assess the goodness-of-fit of a model to the data, we used the Akaike Information Criterion <em xmlns="" class="EmphasisTypeItalic">AIC</em> = 2<em xmlns="" class="EmphasisTypeItalic">k</em> - 2 ln <em xmlns="" class="EmphasisTypeItalic">L</em>, where <em xmlns="" class="EmphasisTypeItalic">k</em> is the number of parameters and <em xmlns="" class="EmphasisTypeItalic">L</em> is the model’s likelihood given the data. When comparing models, the AIC weight of evidence for each model is given by <span class="InlineEquation" id="IEq19"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mi mathvariant="italic">w</mi>
                          <mfenced open="(" close=")">
                            <mi mathvariant="italic">AIC</mi>
                          </mfenced>
                          <mo>∝</mo>
                          <msup>
                            <mi mathvariant="italic">e</mi>
                            <mrow>
                              <mo mathvariant="italic">‒</mo>
                              <mfrac>
                                <mn>1</mn>
                                <mn>2</mn>
                              </mfrac>
                              <mi mathvariant="normal">Δ</mi>
                              <mfenced open="(" close=")">
                                <mi mathvariant="italic">AIC</mi>
                              </mfenced>
                            </mrow>
                          </msup>
                        </mrow>
                      </mrow></semantics></math></span>, where Δ(<em xmlns="" class="EmphasisTypeItalic">AIC</em>) = <em xmlns="" class="EmphasisTypeItalic">AIC</em> ‒ min(<em xmlns="" class="EmphasisTypeItalic">AIC</em>) [<span class="CitationRef"><a href="#CR29">29</a></span>, <span class="CitationRef"><a href="#CR30">30</a></span>].</p>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">We also calculated Pearson’s correlation coefficients between evolutionary rates and the independent variable that defines each model. When comparing two models, we calculated partial correlation coefficients of evolutionary rates with the independent variable of each model controlling that of the other.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec9" class="Section1 RenderAsSection1"><h2 class="Heading js-ToggleCollapseSection">Results and discussion</h2><div class="js-CollapseSection"><p xmlns="http://www.w3.org/1999/xhtml" class="Para">We aim to elucidate whether a site’s rate of evolution depends on flexibility or mutational stress as measured by MSF and MLmS, respectively. To address this issue, for each site of each of the 213 proteins of a dataset of monomeric enzymes, we used the Rate4Site program to estimate its relative evolutionary rate <em xmlns="" class="EmphasisTypeItalic">ω</em><sup><em xmlns="" class="EmphasisTypeItalic">R</em>4<em xmlns="" class="EmphasisTypeItalic">S</em></sup>, we calculated its <em xmlns="" class="EmphasisTypeItalic">MSF</em> using both the pfANM model and the ANM model using Eq. (<span class="InternalRef"><a href="#Equ5">5</a></span>), and we calculated its MLmS for the pfANM and ANM models using Eq. (<span class="InternalRef"><a href="#Equ15">15</a></span>). We also considered as a measure of flexibility the B-factors of the pdb files. As described in Methods, all relative site-specific values were z-score normalized for each protein. All these values for the 77141 sites of the 213 proteins can be found in Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>: Table S2.</p><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">Stress vs. flexibility: protein-by-protein analysis</h3>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">We started by performing a protein-by-protein comparison. For each protein, we fit the competing Stress and Flexibility models to the evolutionary rates <em xmlns="" class="EmphasisTypeItalic">ω</em><sup><em xmlns="" class="EmphasisTypeItalic">R</em>4<em xmlns="" class="EmphasisTypeItalic">S</em></sup>. Results for each protein can be found in Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>: Table S3 and Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>: Table S4. Summary information is shown in Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>. The total AIC (summed over all proteins) is lower for the Stress Model than for the Flexibility model for either ENM potential. The mean AIC weight of evidence is much larger for the Stress Model than for the Flexibility Model. Moreover, mean AIC weights are consistent with counting the number of proteins for which one model outperforms the other: for the pfANM case, the Stress Model is best for 206/213 proteins and for the ANM case for 209/213 proteins. The (absolute value of the) average Pearson correlation is larger for MLmS than for MSF for both ENMs. Moreover, for both ENMs, while the mean partial rate-stress correlations are large, the mean partial rate-flexibility correlations, controlling for stress, are very small. In other words, MSF makes very little independent contribution to the explained variance of site-specific evolutionary rates.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">
                          <strong class="EmphasisTypeBold">Model comparison: protein-by-protein analysis</strong>
                        </p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col align="left"></colgroup><colgroup><col align="left"></colgroup><colgroup><col align="center"></colgroup><colgroup><col align="left"></colgroup><colgroup><col align="center"></colgroup><colgroup><col align="center"></colgroup><colgroup><col align="left"></colgroup><colgroup><col align="left"></colgroup><colgroup><col align="left"></colgroup><thead><tr class="header"><th align="left"><p xmlns="" class="SimplePara">Potential</p></th><th align="left"><p xmlns="" class="SimplePara">Model</p></th><th align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">y</em>
                            </p></th><th align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">x</em>
                            </p></th><th align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">AIC</em>
                            </p></th><th align="center"><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">&lt;w</em>( <em class="EmphasisTypeItalic">AIC</em>)<span class="EmphasisTypeBoldItalic">&gt;</span></p></th><th align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">N</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">prot</em>
                              </sub>
                            </p></th><th align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">&lt;R&gt;</em>
                            </p></th><th align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">&lt;pR&gt;</em>
                            </p></th></tr></thead><tbody><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">pfANM</p></td><td align="left"><p xmlns="" class="SimplePara">Stress</p></td><td align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ω</em>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">MLmS</em>
                            </p></td><td align="center"><p xmlns="" class="SimplePara">190508</p></td><td align="center"><p xmlns="" class="SimplePara">0.97</p></td><td align="left"><p xmlns="" class="SimplePara">206</p></td><td align="left"><p xmlns="" class="SimplePara">-0.54</p></td><td align="left"><p xmlns="" class="SimplePara">-0.33</p></td></tr><tr class="noclass"><td align="left">&nbsp;</td><td align="left"><p xmlns="" class="SimplePara">Flexibility</p></td><td align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ω</em>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">MSF</em>
                            </p></td><td align="center"><p xmlns="" class="SimplePara">198662</p></td><td align="center"><p xmlns="" class="SimplePara">0.03</p></td><td align="left"><p xmlns="" class="SimplePara">7</p></td><td align="left"><p xmlns="" class="SimplePara">0.45</p></td><td align="left"><p xmlns="" class="SimplePara">-0.06</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">ANM</p></td><td align="left"><p xmlns="" class="SimplePara">Stress</p></td><td align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ω</em>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">MLmS</em>
                            </p></td><td align="center"><p xmlns="" class="SimplePara">194154</p></td><td align="center"><p xmlns="" class="SimplePara">0.98</p></td><td align="left"><p xmlns="" class="SimplePara">209</p></td><td align="left"><p xmlns="" class="SimplePara">-0.52</p></td><td align="left"><p xmlns="" class="SimplePara">-0.39</p></td></tr><tr class="noclass"><td align="left">&nbsp;</td><td align="left"><p xmlns="" class="SimplePara">Flexibility</p></td><td align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ω</em>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">MSF</em>
                            </p></td><td align="center"><p xmlns="" class="SimplePara">207258</p></td><td align="center"><p xmlns="" class="SimplePara">0.02</p></td><td align="left"><p xmlns="" class="SimplePara">4</p></td><td align="left"><p xmlns="" class="SimplePara">0.35</p></td><td align="left"><p xmlns="" class="SimplePara">-0.04</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">NOTE: Potential is the ENM potential, Model is either the Stress Model or Flexibility Model, <em class="EmphasisTypeItalic">y</em> is in all cases the site-specific rate of evolution inferred using Rate4Site, <em class="EmphasisTypeItalic">x</em> is the independent variable of each model. <em class="EmphasisTypeItalic">AIC</em> is the Akaike Information Criterion summed over all proteins. &lt;<em class="EmphasisTypeItalic">w(AIC)&gt;</em> is the average of the AIC weight for each compared model (same ENM). <span class="EmphasisTypeBoldItalic">N</span><sub xmlns="http://www.w3.org/1999/xhtml">
                          <span xmlns="" class="EmphasisTypeBoldItalic">prot</span>
                        </sub> is the number of proteins for which each compared model (same ENM) is the best one. &lt;<em class="EmphasisTypeItalic">R&gt;</em> is average over proteins of Pearson’s correlation coefficient between variables <em class="EmphasisTypeItalic">y</em> and <em class="EmphasisTypeItalic">x</em>. &lt;<em class="EmphasisTypeItalic">pR&gt;</em> is the average partial correlation coefficient when controlling for the <em class="EmphasisTypeItalic">x</em> variable of the contrasting model (same ENM). All variables were z-score normalized for each protein before fitting.</p>
                    </div></figure></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">We think that it is most meaningful to compare between MLmS and MSF calculated using the same potential energy landscape (pfANM or ANM). However, the z-normalized MSF values can also be obtained from the B-factors available from the pdb files. We compared the Stress Model, both pfANM-based and ANM-based with a B-factor-based flexibility model and the conclusions are the same (results not shown). In general B-factor based Flexibility Models are the worst (see Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>: Table S3 and Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>: Table S4). This is not surprising because B-factors usually depend very strongly on several factors including experimental conditions, method used to estimate them, crystal disorder, etc. (see [<span class="CitationRef"><a href="#CR31">31</a></span>] and references therein).</p>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">To summarize, whether using the pfANM or the ANM potentials, stress (MLmS) predicts evolutionary rates better than dynamical flexibility (MSF) for almost all proteins of the dataset and the independent contribution of MSF is very small once MLmS is controlled for.</p>
              </section><section id="Sec11" class="Section2 RenderAsSection2">
                <h3 class="Heading">Stress vs. flexibility: global analysis</h3>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">To consider the total evidence in favour of each model we performed a global analysis. We obtained linear fits of the <em xmlns="" class="EmphasisTypeItalic">ω</em><sup><em xmlns="" class="EmphasisTypeItalic">R</em>4<em xmlns="" class="EmphasisTypeItalic">S</em></sup> evolutionary rates to the Stress (Eq.&nbsp;<span class="InternalRef"><a href="#Equ12">12</a></span>) and Flexibility (Eq.&nbsp;<span class="InternalRef"><a href="#Equ6">6</a></span>) models for all 77141 sites of the dataset pooled together. Results are shown in Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>. From AIC it follows that the Stress Model is better than the Flexibility Model for either pfANM or ANM. The Δ(<em xmlns="" class="EmphasisTypeItalic">AIC</em>) values (within the same ENM) are so large that the total weight of evidence for the Stress Model, compared with the Flexibility Model, is <em xmlns="" class="EmphasisTypeItalic">w</em>(<em xmlns="" class="EmphasisTypeItalic">AIC</em>) = 1 whether using pfANM or ANM. The Pearson correlation coefficient R follows the same trend. We note, that even though smaller, the correlation coefficients for rate vs. MSF are significant, which agrees with previous findings [<span class="CitationRef"><a href="#CR14">14</a></span>, <span class="CitationRef"><a href="#CR32">32</a></span>, <span class="CitationRef"><a href="#CR13">13</a></span>, <span class="CitationRef"><a href="#CR15">15</a></span>]. However, partial correlations (pR) show that once stress (MLmS) is controlled for, the rate-MSF correlation almost disappears: the sequence-flexibility correlation is indirect. Similar results are obtained when using B-factors to estimate MSF (results not shown). In summary, the total evidence in support of the Stress Model is very strong.<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">
                          <strong class="EmphasisTypeBold">Model comparison: global analysis</strong>
                        </p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col align="left"></colgroup><colgroup><col align="left"></colgroup><colgroup><col align="center"></colgroup><colgroup><col align="left"></colgroup><colgroup><col align="center"></colgroup><colgroup><col align="center"></colgroup><colgroup><col align="center"></colgroup><colgroup><col align="center"></colgroup><thead><tr class="header"><th align="left"><p xmlns="" class="SimplePara">Potential</p></th><th align="left"><p xmlns="" class="SimplePara">Model</p></th><th align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">y</em>
                            </p></th><th align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">x</em>
                            </p></th><th align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">AIC</em>
                            </p></th><th align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">w(AIC)</em>
                            </p></th><th align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">R</em>
                            </p></th><th align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">pR</em>
                            </p></th></tr></thead><tbody><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">pfANM</p></td><td align="left"><p xmlns="" class="SimplePara">Stress</p></td><td align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ω</em>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">MLmS</em>
                            </p></td><td align="center"><p xmlns="" class="SimplePara">191424</p></td><td align="center"><p xmlns="" class="SimplePara">1.00</p></td><td align="center"><p xmlns="" class="SimplePara">-0.55</p></td><td align="center"><p xmlns="" class="SimplePara">-0.32</p></td></tr><tr class="noclass"><td align="left">&nbsp;</td><td align="left"><p xmlns="" class="SimplePara">Flexibility</p></td><td align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ω</em>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">MSF</em>
                            </p></td><td align="center"><p xmlns="" class="SimplePara">199645</p></td><td align="center"><p xmlns="" class="SimplePara">0.00</p></td><td align="center"><p xmlns="" class="SimplePara">0.47</p></td><td align="center"><p xmlns="" class="SimplePara">-0.04</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">ANM</p></td><td align="left"><p xmlns="" class="SimplePara">Stress</p></td><td align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ω</em>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">MLmS</em>
                            </p></td><td align="center"><p xmlns="" class="SimplePara">194589</p></td><td align="center"><p xmlns="" class="SimplePara">1.00</p></td><td align="center"><p xmlns="" class="SimplePara">-0.52</p></td><td align="center"><p xmlns="" class="SimplePara">-0.40</p></td></tr><tr class="noclass"><td align="left">&nbsp;</td><td align="left"><p xmlns="" class="SimplePara">Flexibility</p></td><td align="center"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ω</em>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">MSF</em>
                            </p></td><td align="center"><p xmlns="" class="SimplePara">207993</p></td><td align="center"><p xmlns="" class="SimplePara">0.00</p></td><td align="center"><p xmlns="" class="SimplePara">0.36</p></td><td align="center"><p xmlns="" class="SimplePara">-0.02</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">NOTE: Results of global fits for all sites of the dataset. Potential is the ENM potential. Model is either the Stress Model or Flexibility Model. <em class="EmphasisTypeItalic">y</em> is in all cases the site-specific rate of evolution inferred using Rate4Site, <em class="EmphasisTypeItalic">x</em> is the independent variable of each model. <em class="EmphasisTypeItalic">AIC</em> is the Akaike Information Criterion, that quantifies the goodness of fit of a model (the lower the better). <em class="EmphasisTypeItalic">w(AIC)</em> is the AIC weight of evidence for each compared model (same ENM). <em class="EmphasisTypeItalic">R</em> is Pearson’s correlation coefficient between variables <em class="EmphasisTypeItalic">y</em> and <em class="EmphasisTypeItalic">x. pR</em> is the partial correlation coefficient when controlling for the <em class="EmphasisTypeItalic">x</em> variable of the contrasting model (same ENM). All variables were z-score normalized for each protein before fitting.</p>
                    </div></figure></div>
              </section><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">Evolutionary rates vs. flexibility and stress</h3>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">What does the dependence of evolutionary rates on flexibility and stress look like? Figure&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span> compares the inferred rates with the predictions of the Stress and Flexibility models. The models were globally fit as described in the previous section. Clearly, the Stress Model fits the inferred rates nicely over almost the whole range, in evident contrast with the Flexibility Model, for both pfANM (top panels) and ANM (bottom panels).<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <img src="https://static-content.springer.com/image/art%3A10.1186%2F1471-2148-14-78/MediaObjects/12862_2014_Article_2554_Fig1_HTML.jpg" alt="">
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 1</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Site-specific evolutionary rates vs. flexibility and stress.</strong> Top panels: parameter-free Anisotropic Network Model (pfANM). Bottom panels: Anisotropic Network Model. Black circles: site-specific rates inferred from the sequence alignments using Rate4Site. Vertical segments: 99% confidence interval of the circles. Solid line: Stress Model predictions. Dashed line: Flexibility Model predictions. For the rate vs. flexibility plots (left panels), sites were evenly split between 20 bins according to their Mean Square Fluctuation (MSF) dynamical flexibility measure, and then MSF and evolutionary rates were averaged within bins. For the rate vs. stress plots (right panels), sites were evenly split into bins according to their Mean Local mutational Stress (MLmS), and average MLmS and rates were obtained for each bin. All variables were z-normalized for each protein.</p>
                      </div></figcaption></figure></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Even though previous sequence-flexibility studies used Pearson correlations, which, rigorously, make sense only for linear relationships, they already found nonlinear sequence-flexibility plots similar to those of Figure&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span> (left panels) [<span class="CitationRef"><a href="#CR14">14</a></span>, <span class="CitationRef"><a href="#CR32">32</a></span>, <span class="CitationRef"><a href="#CR13">13</a></span>]. In spite of this, they either dismissed the nonlinear part [<span class="CitationRef"><a href="#CR14">14</a></span>]or interpreted it in terms of different selection regimes [<span class="CitationRef"><a href="#CR13">13</a></span>]. From Figure&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span> (left panels) it is clear that the nonlinearity follows naturally from the proposed Stress Model, suggesting that evolutionary rates depend nonlinearly on <em xmlns="" class="EmphasisTypeItalic">MSF</em> because they depend (approximately) linearly on <em xmlns="" class="EmphasisTypeItalic">MLmS</em>, and <em xmlns="" class="EmphasisTypeItalic">MSF</em> ≈ 1/<em xmlns="" class="EmphasisTypeItalic">MLmS</em>, which can be derived from Eqs. (<span class="InternalRef"><a href="#Equ19">19</a></span>) and (<span class="InternalRef"><a href="#Equ20">20</a></span>).</p>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">To conclude this subsection, we must observe that inferred rates are larger than stress-based predictions for the slowest sites and smaller for the fastest. A reason could be that inference methods overestimate small rates and underestimate large ones [<span class="CitationRef"><a href="#CR33">33</a></span>]. However, close inspection of the rate vs. stress curves (right panels of Figure&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>) indicates that despite the very good fit of the linear Stress Model, there still seems to be some remaining nonlinearity of the <em xmlns="" class="EmphasisTypeItalic">ω</em><sup><em xmlns="" class="EmphasisTypeItalic">R</em>4<em xmlns="" class="EmphasisTypeItalic">S</em></sup> vs. <em xmlns="" class="EmphasisTypeItalic">MLmS</em> plots. A possible reason is the weak-selection approximation used to linearize the exponential in Eq. (<span class="InternalRef"><a href="#Equ11">11</a></span>), however, resolving this issue is beyond the scope of the present report.</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">pfANM vs. ANM</h3>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">To finish this section, we compare ANM with pfANM. Figure&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span> shows that both pfANM and ANM result in similar qualitative dependence of rate vs. flexibility (left panels) and rate vs. stress (right panels). However, the pfANM potential (top panels) results in better fits to the inferred rates than the ANM potential (bottom panels). Accordingly, the AIC values (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span> and Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>) show that the pfANM-based stress model is better than the one based on ANM. This is in agreement with the finding that WCN correlates better than CN with evolutionary rates [<span class="CitationRef"><a href="#CR11">11</a></span>].</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec14" class="Section1 RenderAsSection1"><h2 class="Heading js-ToggleCollapseSection">Conclusion</h2><div class="js-CollapseSection"><p xmlns="http://www.w3.org/1999/xhtml" class="Para">We introduced a mechanistic Stress Model of protein sequence evolution. Mutations are modelled as random perturbations of the protein’s potential energy landscape, represented using Elastic Network Models. To model natural selection, we used basic statistical physics to derive the expected probability that a mutant samples a specific functional structure. From this, we deduced a linear relationship between a site’s mean evolutionary rate and the mean local mutational stress (MLmS) of the functional conformation. We compared this model with an empirical Flexibility Model that postulates that a site’s evolutionary rate is linearly dependent on its flexibility (measured by its MSF). We compared both models and found strong support for the Stress Model. Moreover, the independent contribution of flexibility is negligible once stress is controlled for.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">The MLmS is proportional to Local Packing Density and, therefore, the Stress Model provides a mechanism for the connection between a site’s LPD and its evolutionary rate. Regarding the sequence-flexibility relationship, previous empirical correlation studies had already found that the sequence-flexibility relationship is nonlinear and either dismissed the nonlinear parts or attempted an interpretation in terms of different selection regimes [<span class="CitationRef"><a href="#CR14">14</a></span>, <span class="CitationRef"><a href="#CR32">32</a></span>, <span class="CitationRef"><a href="#CR13">13</a></span>]. We found the nonlinearity follows naturally from the Stress Model: evolutionary rates depend nonlinearly on <em xmlns="" class="EmphasisTypeItalic">MSF</em> because they depend (approximately) linearly on <em xmlns="" class="EmphasisTypeItalic">MLmS</em>, and <em xmlns="" class="EmphasisTypeItalic">MSF</em> ≈ 1/<em xmlns="" class="EmphasisTypeItalic">MLmS</em>. To summarize, the Stress Model accounts for the observed site-dependency of evolutionary rates and its relationship with packing density and flexibility.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">A note of caution is in order here. For the Stress Model mutational stress was not postulated to be the determinant factor <em xmlns="" class="EmphasisTypeItalic">a priori</em> but, rather, it was derived from the assumptions of the model that are essentially two (1) there is a single active conformation and (2) mutants are flexible and therefore can sample the active conformation so that they are at least partly functional. Therefore even though Stress Model was chosen to designate this mechanistic model, it should be kept in mind that it demonstrates the importance of protein flexibility.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">It is worthwhile to mention some of the possible caveats and further developments of the Stress Model. First, we assume a single active conformation. In principle, it would be reasonable to assume that only changes of the active-site conformation should affect fitness. However, we note that if protein sites are strongly coupled, which is often the case, any conformational change will affect the active site conformation. For a strongly coupled elastic network forcing the active site to adopt a given conformation makes the rest of the protein move accordingly. Therefore, assuming that the whole protein conformation must be in the “active conformation” for the protein to function is not necessarily an important limitation. However, for cases where the coupling is not very strong, if the active site is known, this could be easily tackled using a modified version of the selection function that integrates away all coordinates except for those of the active site (i.e. uses marginal conformational distributions rather than the full ones in the definition of selection function).</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">Second, in Eq. (<span class="InternalRef"><a href="#Equ11">11</a></span>) we performed a linear approximation of the exponential function. This is reasonable <em xmlns="" class="EmphasisTypeItalic">a priori</em> only for weak selection, and <em xmlns="" class="EmphasisTypeItalic">a posteriori</em> by the good fit of the resulting model to the data. We should note, however, this approximation can be easily removed, and the actual mean of the exponential can be calculated via simulation. Further work is needed to explore this possibility.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">Third, we note that the z-normalized MLmS values, on which the Stress Model is based, are identical to the z-normalized LPD measures WCN (for the pfANM potential) and CN (for the ANM potential). For other potentials this need not be the case and it is for that reason that we chose to keep the notation MLmS in the present tables and figures, to make them comparable with further research based on estimating MLmS using different, perhaps better, potential energy functions.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">To close, we note that the mutational part of the Stress Model accounts for observed patterns of evolutionary divergence of protein structure and dynamics [<span class="CitationRef"><a href="#CR21">21</a></span>–<span class="CitationRef"><a href="#CR23">23</a></span>]. Regarding structural divergence, unselected random mutations reproduce very well the evolutionary conservation of a “structural core” and account for the observation that structures diverge mainly within the space spanned by a few low-energy collective normal modes [<span class="CitationRef"><a href="#CR21">21</a></span>, <span class="CitationRef"><a href="#CR22">22</a></span>]. Regarding protein motions, unselected random mutations explain the higher conservation of the low-energy normal modes in terms of their mutational robustness [<span class="CitationRef"><a href="#CR31">31</a></span>, <span class="CitationRef"><a href="#CR23">23</a></span>]. In general, those studies could found no evidence of natural selection at the levels of structural or dynamical divergence. Clearly, without natural selection, all sites would evolve at the same rate, which is not the case. The Stress Model proposed here accounts rather well for the variation of rates of evolution among sites. It would be interesting to study the effect of the selection function introduced here on structural and dynamical divergence and compare the observed patterns with those that result from unselected mutations. This could advance our understanding of the effect of selection at the levels of structure and dynamics. In general, we think the Stress Model provides a possible unifying framework to study evolutionary protein divergence at the levels of sequence, structure, and dynamics.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 ArticleNotes" id="ArticleNotes"><h2 class="Heading js-ToggleCollapseSection">Notes</h2><div class="js-CollapseSection"><aside xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><p xmlns="" class="SimplePara">Tsun-Tsao Huang, María Laura del Valle Marcos contributed equally to this work.</p></aside></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup"><h2 class="Heading js-ToggleCollapseSection">Abbreviations</h2><div class="js-CollapseSection"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>RSA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Relative solvent accessibility</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>LPD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Local packing density</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>WCN:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Weighted contact number</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>CN:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Contact number</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>ENM:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Elastic network model</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>ANM:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Anisotropic network model</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>pfANM:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns="http://www.w3.org/1999/xhtml" class="Para">parameter-free anisotropic network model</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>MSF:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Mean square fluctuation</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>MLmS:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Mean local mutational stress</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>AIC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Akaike information criterion.</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1" id="Declarations"><h2 class="Heading js-ToggleCollapseSection">
              Declarations
            </h2><div class="js-CollapseSection"><div xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">This research was supported in part by Academic Summit Program of National Science Council with grant number NSC-102-2745-B-009-001 and the “Center for Bioinformatics Research of Aiming for the Top University Program” of the National Chiao Tung University and Ministry of Education, Taiwan, R.O.C. JE is a researcher of CONICET.</p></div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Electronic supplementary material</h3><div id="widgetContainer"></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2148-14-78/MediaObjects/12862_2014_2554_MOESM1_ESM.csv">12862_2014_2554_MOESM1_ESM.csv</a>
                          <span xmlns="" class="SimplePara">Additional file 1: Table S1: Protein Dataset. Protein: pdb code; chain: protein chain; EC.class: Enzyme Comission class; SCOP class: structural class of domains according to the SCOP classification; Domains: number of domains; Year: year in which the structure was determined; Resolution: resolution of the X-ray experiment; Sites: number of sites of the protein chain; Number.of.Sequences: number of sequences of the multiple sequence alignment used for site-specific rate inferences. (CSV 13 KB)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2148-14-78/MediaObjects/12862_2014_2554_MOESM2_ESM.csv">12862_2014_2554_MOESM2_ESM.csv</a>
                          <span xmlns="" class="SimplePara">Additional file 2: Table S2: Site-specific rates, flexibility, and stress measures. pdb: pdb identifier of the protein; chain: protein chain; site: protein site; zwr4s: z-normalized site-specific rate of evolution inferred using Rate4Site; zbfactor: z-normalized B-factor; zmsf_pfanm: z-normalized Mean Square Fluctuation (MSF) obtained with the pfANM model; zmlms_pfanm: z-normalized Mean Local mutational Stress (MLmS) obtained with the pfANM model; zmsf_anm: z-normalized MSF for the ANM model; zmlms_anm: z-normalized MLmS for the ANM model. (CSV 9 MB)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2148-14-78/MediaObjects/12862_2014_2554_MOESM3_ESM.csv">12862_2014_2554_MOESM3_ESM.csv</a>
                          <span xmlns="" class="SimplePara">Additional file 3: Table S3.: Protein by protein goodness-of-fit measures. pdb: pdb identifier of the protein; chain: protein chain; aic.zmlms_pfanm: Akaike Information Criterion (AIC) of the pfANM-based Stress Model; aic.zmsf_pfanm: AIC of the pfANM-based Flexibility Model; aic.zmlms_anm: AIC for the ANM-based Stress Model; aic.zmsf_anm: AIC for the ANM-based Flexibility Model: AIC.zbfactor: AIC for a B-factor based Flexibility Model; r.zmlms_pfanm: Pearson correlation coefficient (R) of the pfANM-based Stress Model; r.zmsf_pfanm: R of the pfANM-based Flexibility Model; r.zmlms_anm: R for the ANM-based Stress Model; r.zmsf_anm: R for the ANM-based Flexibility Model; r.zbfactor: R for a B-factor-based Flexibility Model. (CSV 39 KB)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2148-14-78/MediaObjects/12862_2014_2554_MOESM4_ESM.csv">12862_2014_2554_MOESM4_ESM.csv</a>
                          <span xmlns="" class="SimplePara">Additional file 4: Table S4: Protein by protein comparison of pairs of models. pdb: pdb identifier of the protein; chain: protein chain; waic.m1.m2 is the Akaike Information Criterion weight of evidence w(AIC) of m1 in an m1 vs m2 comparison. pr.m1.m2 is the partial correlation of site-specific rates of evolution with the independent variable defining m1, controlling that of m2. Models considers are: pfANM-based Stress Model (zmlms_pfanm); pfANM-based Flexibility Model (zmsf_pfanm); ANM-based Stress Model (zmlms_anm); ANM-based Flexibility Model (zmsf_anm); B-factor-based Flexibility Model (zbfactor). (CSV 78 KB)</span>
                        </div></div>
                  </div>
                </div>
              
              
                
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">Below are the links to the authors’ original submitted files for images.<div class="MediaObject" id="MOESM5">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2148-14-78/MediaObjects/12862_2014_2554_MOESM5_ESM.pdf">12862_2014_2554_MOESM5_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 1</span>
                        </div></div>
                  </div></div>
              
            </aside><aside xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Competing interests</h3><p xmlns="" class="SimplePara">The authors declare that they have no competing interests.</p></aside><aside xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">JE and JKH designed the study. JE derived the Stress Model. TTH and MLVM wrote the codes and performed the calculations and analysed data. JE drafted the manuscript. All authors commented on and approved the final manuscript.</p></aside></div></section></div>
                            <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection">Authors’ Affiliations</h2><div class="js-CollapseSection"><div class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Institute of Bioinformatics and Systems Biology, National Chiao Tung University</div></div><div class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Center for Bioinformatics Research, National Chiao Tung University</div></div><div class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">Escuela de Ciencia y Tecnología, Universidad Nacional de San Martín</div></div></div></section></div>
                            <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection">References</h2><div class="js-CollapseSection"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1">Pal C, Papp B, Lercher MJ: An integrated view of protein evolution. Nat Rev Genet. 2006, 7 (5): 337-348. 10.1038/nrg1838.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16619049"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1038/nrg1838"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=An%20integrated%20view%20of%20protein%20evolution&amp;author=C.%20Pal&amp;author=B.%20Papp&amp;author=MJ.%20Lercher&amp;journal=Nat%20Rev%20Genet&amp;volume=7&amp;issue=5&amp;pages=337-348&amp;publication_year=2006&amp;doi=10.1038%2Fnrg1838"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2">Thorne JL: Protein evolution constraints and model-based techniques to study them. Curr Opin Struct Biol. 2007, 17 (3): 337-341. 10.1016/j.sbi.2007.05.006.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17572082"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.sbi.2007.05.006"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Protein%20evolution%20constraints%20and%20model-based%20techniques%20to%20study%20them&amp;author=JL.%20Thorne&amp;journal=Curr%20Opin%20Struct%20Biol&amp;volume=17&amp;issue=3&amp;pages=337-341&amp;publication_year=2007&amp;doi=10.1016%2Fj.sbi.2007.05.006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3">Worth CL, Gong S, Blundell TL: Structural and functional constraints in the evolution of protein families. Nat Rev Mol Cell Biol. 2009, 10 (10): 709-720.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19756040"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Structural%20and%20functional%20constraints%20in%20the%20evolution%20of%20protein%20families&amp;author=CL.%20Worth&amp;author=S.%20Gong&amp;author=TL.%20Blundell&amp;journal=Nat%20Rev%20Mol%20Cell%20Biol&amp;volume=10&amp;issue=10&amp;pages=709-720&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4">Wilke CO, Drummond DA: Signatures of protein biophysics in coding sequence evolution. Curr Opin Struct Biol. 2010, 20 (3): 385-389. 10.1016/j.sbi.2010.03.004.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20395125"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2905607"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.sbi.2010.03.004"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Signatures%20of%20protein%20biophysics%20in%20coding%20sequence%20evolution&amp;author=CO.%20Wilke&amp;author=DA.%20Drummond&amp;journal=Curr%20Opin%20Struct%20Biol&amp;volume=20&amp;issue=3&amp;pages=385-389&amp;publication_year=2010&amp;doi=10.1016%2Fj.sbi.2010.03.004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5">Grahnen JA, Nandakumar P, Kubelka J, Liberles DA: Biophysical and structural considerations for protein sequence evolution. BMC Evol Biol. 2011, 11 (1): 361-10.1186/1471-2148-11-361.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22171550"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3292521"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1186/1471-2148-11-361"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Biophysical%20and%20structural%20considerations%20for%20protein%20sequence%20evolution&amp;author=JA.%20Grahnen&amp;author=P.%20Nandakumar&amp;author=J.%20Kubelka&amp;author=DA.%20Liberles&amp;journal=BMC%20Evol%20Biol&amp;volume=11&amp;issue=1&amp;pages=361&amp;publication_year=2011&amp;doi=10.1186%2F1471-2148-11-361"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6">Liberles DA, Teichmann SA, Bahar I, Bastolla U, Bloom J, Bornberg-Bauer E, Colwell LJ, de Koning AP, Dokholyan NV, Echave J, Elofsson A, Gerloff DL, Goldstein RA, Grahnen JA, Holder MT, Lakner C, Lartillot N, Lovell SC, Naylor G, Perica T, Pollock DD, Pupko T, Regan L, Roger A, Rubinstein N, Shakhnovich E, Sjolander K, Sunyaev S, Teufel AI, Thorne JL, et al: The interface of protein structure, protein biophysics, and molecular evolution. Protein Sci. 2012, 21 (6): 769-785. 10.1002/pro.2071.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22528593"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3403413"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1002/pro.2071"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=The%20interface%20of%20protein%20structure%2C%20protein%20biophysics%2C%20and%20molecular%20evolution&amp;author=DA.%20Liberles&amp;author=SA.%20Teichmann&amp;author=I.%20Bahar&amp;author=U.%20Bastolla&amp;author=J.%20Bloom&amp;author=E.%20Bornberg-Bauer&amp;author=LJ.%20Colwell&amp;author=AP.%20de%20Koning&amp;author=NV.%20Dokholyan&amp;author=J.%20Echave&amp;author=A.%20Elofsson&amp;author=DL.%20Gerloff&amp;author=RA.%20Goldstein&amp;author=JA.%20Grahnen&amp;author=MT.%20Holder&amp;author=C.%20Lakner&amp;author=N.%20Lartillot&amp;author=SC.%20Lovell&amp;author=G.%20Naylor&amp;author=T.%20Perica&amp;author=DD.%20Pollock&amp;author=T.%20Pupko&amp;author=L.%20Regan&amp;author=A.%20Roger&amp;author=N.%20Rubinstein&amp;author=E.%20Shakhnovich&amp;author=K.%20Sjolander&amp;author=S.%20Sunyaev&amp;author=AI.%20Teufel&amp;author=JL.%20Thorne&amp;journal=Protein%20Sci&amp;volume=21&amp;issue=6&amp;pages=769-785&amp;publication_year=2012&amp;doi=10.1002%2Fpro.2071"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7">Bustamante CD, Townsend JP, Hartl DL: Solvent accessibility and purifying selection within proteins of Escherichia coli and Salmonella enterica. Mol Biol Evol. 2000, 17 (2): 301-308. 10.1093/oxfordjournals.molbev.a026310.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10677853"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026310"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Solvent%20accessibility%20and%20purifying%20selection%20within%20proteins%20of%20Escherichia%20coli%20and%20Salmonella%20enterica&amp;author=CD.%20Bustamante&amp;author=JP.%20Townsend&amp;author=DL.%20Hartl&amp;journal=Mol%20Biol%20Evol&amp;volume=17&amp;issue=2&amp;pages=301-308&amp;publication_year=2000&amp;doi=10.1093%2Foxfordjournals.molbev.a026310"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8">Dean AM, Neuhauser C, Grenier E, Golding GB: The pattern of amino acid replacements in alpha/beta-barrels. Mol Biol Evol. 2002, 19 (11): 1846-1864. 10.1093/oxfordjournals.molbev.a004009.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12411594"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004009"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=The%20pattern%20of%20amino%20acid%20replacements%20in%20alpha%2Fbeta-barrels&amp;author=AM.%20Dean&amp;author=C.%20Neuhauser&amp;author=E.%20Grenier&amp;author=GB.%20Golding&amp;journal=Mol%20Biol%20Evol&amp;volume=19&amp;issue=11&amp;pages=1846-1864&amp;publication_year=2002&amp;doi=10.1093%2Foxfordjournals.molbev.a004009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9">Franzosa EA, Xia Y: Structural determinants of protein evolution are context-sensitive at the residue level. Mol Biol Evol. 2009, 26 (10): 2387-2395. 10.1093/molbev/msp146.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19597162"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1093/molbev/msp146"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Structural%20determinants%20of%20protein%20evolution%20are%20context-sensitive%20at%20the%20residue%20level&amp;author=EA.%20Franzosa&amp;author=Y.%20Xia&amp;journal=Mol%20Biol%20Evol&amp;volume=26&amp;issue=10&amp;pages=2387-2395&amp;publication_year=2009&amp;doi=10.1093%2Fmolbev%2Fmsp146"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10">Ramsey DC, Scherrer MP, Zhou T, Wilke CO: The relationship between relative solvent accessibility and evolutionary rate in protein evolution. Genetics. 2011, 188 (2): 479-488. 10.1534/genetics.111.128025.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21467571"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3122320"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1534/genetics.111.128025"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=The%20relationship%20between%20relative%20solvent%20accessibility%20and%20evolutionary%20rate%20in%20protein%20evolution&amp;author=DC.%20Ramsey&amp;author=MP.%20Scherrer&amp;author=T.%20Zhou&amp;author=CO.%20Wilke&amp;journal=Genetics&amp;volume=188&amp;issue=2&amp;pages=479-488&amp;publication_year=2011&amp;doi=10.1534%2Fgenetics.111.128025"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11">Yeh S-W, Liu J-W, Yu S-H, Shih C-H, Hwang J-K, Echave J: Site-specific structural constraints on protein sequence evolutionary divergence: local packing density versus solvent exposure. Mol Biol Evol. 2014, 31 (1): 135-139. 10.1093/molbev/mst178.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24109601"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1093/molbev/mst178"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Site-specific%20structural%20constraints%20on%20protein%20sequence%20evolutionary%20divergence%3A%20local%20packing%20density%20versus%20solvent%20exposure&amp;author=S-W.%20Yeh&amp;author=J-W.%20Liu&amp;author=S-H.%20Yu&amp;author=C-H.%20Shih&amp;author=J-K.%20Hwang&amp;author=J.%20Echave&amp;journal=Mol%20Biol%20Evol&amp;volume=31&amp;issue=1&amp;pages=135-139&amp;publication_year=2014&amp;doi=10.1093%2Fmolbev%2Fmst178"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12">Halle B: Flexibility and packing in proteins. Proc Natl Acad Sci U S A. 2002, 99 (3): 1274-1279. 10.1073/pnas.032522499.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11818549"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC122180"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1073/pnas.032522499"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Flexibility%20and%20packing%20in%20proteins&amp;author=B.%20Halle&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=99&amp;issue=3&amp;pages=1274-1279&amp;publication_year=2002&amp;doi=10.1073%2Fpnas.032522499"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13">Liu Y, Bahar I: Sequence evolution correlates with structural dynamics. Mol Biol Evol. 2012, 29 (9): 2253-2263. 10.1093/molbev/mss097.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22427707"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3424413"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1093/molbev/mss097"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Sequence%20evolution%20correlates%20with%20structural%20dynamics&amp;author=Y.%20Liu&amp;author=I.%20Bahar&amp;journal=Mol%20Biol%20Evol&amp;volume=29&amp;issue=9&amp;pages=2253-2263&amp;publication_year=2012&amp;doi=10.1093%2Fmolbev%2Fmss097"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14">Liao H, Yeh W, Chiang D, Jernigan RL, Lustig B: Protein sequence entropy is closely related to packing density and hydrophobicity. Protein Eng Des Sel. 2005, 18 (2): 59-64. 10.1093/protein/gzi009.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15788422"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2553042"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1093/protein/gzi009"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Protein%20sequence%20entropy%20is%20closely%20related%20to%20packing%20density%20and%20hydrophobicity&amp;author=H.%20Liao&amp;author=W.%20Yeh&amp;author=D.%20Chiang&amp;author=RL.%20Jernigan&amp;author=B.%20Lustig&amp;journal=Protein%20Eng%20Des%20Sel&amp;volume=18&amp;issue=2&amp;pages=59-64&amp;publication_year=2005&amp;doi=10.1093%2Fprotein%2Fgzi009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15">Shih C-H, Chang C-M, Lin Y-S, Lo W-C, Hwang J-K: Evolutionary information hidden in a single protein structure. Proteins. 2012, 80 (6): 1647-1657. 10.1002/prot.24058.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22454236"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1002/prot.24058"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Evolutionary%20information%20hidden%20in%20a%20single%20protein%20structure&amp;author=C-H.%20Shih&amp;author=C-M.%20Chang&amp;author=Y-S.%20Lin&amp;author=W-C.%20Lo&amp;author=J-K.%20Hwang&amp;journal=Proteins&amp;volume=80&amp;issue=6&amp;pages=1647-1657&amp;publication_year=2012&amp;doi=10.1002%2Fprot.24058"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16">Nevin Gerek Z, Kumar S, Banu Ozkan S: Structural dynamics flexibility informs function and evolution at a proteome scale. Evol Appl. 2013, 6 (3): 423-433. 10.1111/eva.12052.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23745135"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3673471"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1111/eva.12052"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Structural%20dynamics%20flexibility%20informs%20function%20and%20evolution%20at%20a%20proteome%20scale&amp;author=Z.%20Nevin%20Gerek&amp;author=S.%20Kumar&amp;author=S.%20Banu%20Ozkan&amp;journal=Evol%20Appl&amp;volume=6&amp;issue=3&amp;pages=423-433&amp;publication_year=2013&amp;doi=10.1111%2Feva.12052"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17">Bahar I, Lezon TR, Yang LW, Eyal E: Global dynamics of proteins: bridging between structure and function. Annu Rev Biophys. 2010, 39: 23-42. 10.1146/annurev.biophys.093008.131258.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20192781"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2938190"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1146/annurev.biophys.093008.131258"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Global%20dynamics%20of%20proteins%3A%20bridging%20between%20structure%20and%20function&amp;author=I.%20Bahar&amp;author=TR.%20Lezon&amp;author=LW.%20Yang&amp;author=E.%20Eyal&amp;journal=Annu%20Rev%20Biophys&amp;volume=39&amp;pages=23-42&amp;publication_year=2010&amp;doi=10.1146%2Fannurev.biophys.093008.131258"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18">Micheletti C: Comparing proteins by their internal dynamics: exploring structure–function relationships beyond static structural alignments. Phys Life Rev. 2013, 10 (1): 1-26. 10.1016/j.plrev.2012.10.009.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23199577"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.plrev.2012.10.009"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Comparing%20proteins%20by%20their%20internal%20dynamics%3A%20exploring%20structure%E2%80%93function%20relationships%20beyond%20static%20structural%20alignments&amp;author=C.%20Micheletti&amp;journal=Phys%20Life%20Rev&amp;volume=10&amp;issue=1&amp;pages=1-26&amp;publication_year=2013&amp;doi=10.1016%2Fj.plrev.2012.10.009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19">Sanejouand YH: Elastic network models: theoretical and empirical foundations. Methods Mol Biol. 2013, 924: 601-616. 10.1007/978-1-62703-017-5_23.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23034765"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1007/978-1-62703-017-5_23"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Elastic%20network%20models%3A%20theoretical%20and%20empirical%20foundations&amp;author=YH.%20Sanejouand&amp;journal=Methods%20Mol%20Biol&amp;volume=924&amp;pages=601-616&amp;publication_year=2013&amp;doi=10.1007%2F978-1-62703-017-5_23"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20">Atilgan AR, Durell SR, Jernigan RL, Demirel MC, Keskin O, Bahar I: Anisotropy of fluctuation dynamics of proteins with an elastic network model. Biophys J. 2001, 80 (1): 505-515. 10.1016/S0006-3495(01)76033-X.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11159421"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1301252"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/S0006-3495(01)76033-X"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Anisotropy%20of%20fluctuation%20dynamics%20of%20proteins%20with%20an%20elastic%20network%20model&amp;author=AR.%20Atilgan&amp;author=SR.%20Durell&amp;author=RL.%20Jernigan&amp;author=MC.%20Demirel&amp;author=O.%20Keskin&amp;author=I.%20Bahar&amp;journal=Biophys%20J&amp;volume=80&amp;issue=1&amp;pages=505-515&amp;publication_year=2001&amp;doi=10.1016%2FS0006-3495%2801%2976033-X"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21">Echave J: Evolutionary divergence of protein structure: the linearly forced elastic network model. Chem Phys Lett. 2008, 457 (4–6): 413-416.<span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.cplett.2008.04.042"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Evolutionary%20divergence%20of%20protein%20structure%3A%20the%20linearly%20forced%20elastic%20network%20model&amp;author=J.%20Echave&amp;journal=Chem%20Phys%20Lett&amp;volume=457&amp;issue=4%E2%80%936&amp;pages=413-416&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22">Echave J, Fernandez FM: A perturbative view of protein structural variation. Proteins. 2010, 78 (1): 173-180. 10.1002/prot.22553.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19731380"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1002/prot.22553"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=A%20perturbative%20view%20of%20protein%20structural%20variation&amp;author=J.%20Echave&amp;author=FM.%20Fernandez&amp;journal=Proteins&amp;volume=78&amp;issue=1&amp;pages=173-180&amp;publication_year=2010&amp;doi=10.1002%2Fprot.22553"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23">Echave J: Why are the low-energy protein normal modes evolutionarily conserved?. Pure Appl Chem. 2012, 84 (9): 1931-1937.<span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1351/PAC-CON-12-02-15"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Why%20are%20the%20low-energy%20protein%20normal%20modes%20evolutionarily%20conserved%3F&amp;author=J.%20Echave&amp;journal=Pure%20Appl%20Chem&amp;volume=84&amp;issue=9&amp;pages=1931-1937&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24">Fuglebakk E, Reuter N, Hinsen K: Evaluation of protein elastic network models based on an analysis of collective motions. J Chem Theory Comput. 2013, 9 (12): 5618-5628. 10.1021/ct400399x.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26592296"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1021/ct400399x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Evaluation%20of%20protein%20elastic%20network%20models%20based%20on%20an%20analysis%20of%20collective%20motions&amp;author=E.%20Fuglebakk&amp;author=N.%20Reuter&amp;author=K.%20Hinsen&amp;journal=J%20Chem%20Theory%20Comput&amp;volume=9&amp;issue=12&amp;pages=5618-5628&amp;publication_year=2013&amp;doi=10.1021%2Fct400399x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25">Yang L, Song G, Jernigan RL: Protein elastic network models and the ranges of cooperativity. Proc Natl Acad Sci U S A. 2009, 106 (30): 12347-12352. 10.1073/pnas.0902159106.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19617554"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2718344"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1073/pnas.0902159106"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Protein%20elastic%20network%20models%20and%20the%20ranges%20of%20cooperativity&amp;author=L.%20Yang&amp;author=G.%20Song&amp;author=RL.%20Jernigan&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=106&amp;issue=30&amp;pages=12347-12352&amp;publication_year=2009&amp;doi=10.1073%2Fpnas.0902159106"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26">Lin C-P, Huang S-W, Lai Y-L, Yen S-C, Shih C-H, Lu C-H, Huang C-C, Hwang J-K: Deriving protein dynamical properties from weighted protein contact number. Proteins. 2008, 72 (3): 929-935. 10.1002/prot.21983.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18300253"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1002/prot.21983"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Deriving%20protein%20dynamical%20properties%20from%20weighted%20protein%20contact%20number&amp;author=C-P.%20Lin&amp;author=S-W.%20Huang&amp;author=Y-L.%20Lai&amp;author=S-C.%20Yen&amp;author=C-H.%20Shih&amp;author=C-H.%20Lu&amp;author=C-C.%20Huang&amp;author=J-K.%20Hwang&amp;journal=Proteins&amp;volume=72&amp;issue=3&amp;pages=929-935&amp;publication_year=2008&amp;doi=10.1002%2Fprot.21983"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27">Pupko T, Bell RE, Mayrose I, Glaser F, Ben-Tal N: Rate4Site: an algorithmic tool for the identification of functional regions in proteins by surface mapping of evolutionary determinants within their homologues. Bioinformatics. 2002, 18 (Suppl 1): S71-S77. 10.1093/bioinformatics/18.suppl_1.S71.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12169533"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/18.suppl_1.S71"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Rate4Site%3A%20an%20algorithmic%20tool%20for%20the%20identification%20of%20functional%20regions%20in%20proteins%20by%20surface%20mapping%20of%20evolutionary%20determinants%20within%20their%20homologues&amp;author=T.%20Pupko&amp;author=RE.%20Bell&amp;author=I.%20Mayrose&amp;author=F.%20Glaser&amp;author=N.%20Ben-Tal&amp;journal=Bioinformatics&amp;volume=18&amp;issue=Suppl%201&amp;pages=S71-S77&amp;publication_year=2002&amp;doi=10.1093%2Fbioinformatics%2F18.suppl_1.S71"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28">Mayrose I, Graur D, Ben-Tal N, Pupko T: Comparison of site-specific rate-inference methods for protein sequences: empirical Bayesian methods are superior. Mol Biol Evol. 2004, 21 (9): 1781-1791. 10.1093/molbev/msh194.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15201400"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1093/molbev/msh194"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20site-specific%20rate-inference%20methods%20for%20protein%20sequences%3A%20empirical%20Bayesian%20methods%20are%20superior&amp;author=I.%20Mayrose&amp;author=D.%20Graur&amp;author=N.%20Ben-Tal&amp;author=T.%20Pupko&amp;journal=Mol%20Biol%20Evol&amp;volume=21&amp;issue=9&amp;pages=1781-1791&amp;publication_year=2004&amp;doi=10.1093%2Fmolbev%2Fmsh194"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29">Wagenmakers EJ, Farrell S: AIC model selection using Akaike weights. Psychon Bull Rev. 2004, 11 (1): 192-196. 10.3758/BF03206482.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15117008"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.3758/BF03206482"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=AIC%20model%20selection%20using%20Akaike%20weights&amp;author=EJ.%20Wagenmakers&amp;author=S.%20Farrell&amp;journal=Psychon%20Bull%20Rev&amp;volume=11&amp;issue=1&amp;pages=192-196&amp;publication_year=2004&amp;doi=10.3758%2FBF03206482"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30">Spiess AN, Neumeyer N: An evaluation of R2 as an inadequate measure for nonlinear models in pharmacological and biochemical research: a Monte Carlo approach. BMC Pharmacol. 2010, 10: 6-<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20529254"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2892436"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1186/1471-2210-10-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=An%20evaluation%20of%20R2%20as%20an%20inadequate%20measure%20for%20nonlinear%20models%20in%20pharmacological%20and%20biochemical%20research%3A%20a%20Monte%20Carlo%20approach&amp;author=AN.%20Spiess&amp;author=N.%20Neumeyer&amp;journal=BMC%20Pharmacol&amp;volume=10&amp;pages=6&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31">Maguid S, Fernandez-Alberti S, Echave J: Evolutionary conservation of protein vibrational dynamics. Gene. 2008, 422 (1–2): 7-13.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18577430"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.gene.2008.06.002"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Evolutionary%20conservation%20of%20protein%20vibrational%20dynamics&amp;author=S.%20Maguid&amp;author=S.%20Fernandez-Alberti&amp;author=J.%20Echave&amp;journal=Gene&amp;volume=422&amp;issue=1%E2%80%932&amp;pages=7-13&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32">Jernigan RL, Kloczkowski A: Packing regularities in biological structures relate to their dynamics. Methods Mol Biol. 2007, 350: 251-276.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16957327"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2039702"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Packing%20regularities%20in%20biological%20structures%20relate%20to%20their%20dynamics&amp;author=RL.%20Jernigan&amp;author=A.%20Kloczkowski&amp;journal=Methods%20Mol%20Biol&amp;volume=350&amp;pages=251-276&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33">Fernandes AD, Atchley WR: Site-specific evolutionary rates in proteins are better modeled as non-independent and strictly relative. Bioinformatics. 2008, 24 (19): 2177-2183. 10.1093/bioinformatics/btn395.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18662926"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2553437"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1093/bioinformatics/btn395"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Site-specific%20evolutionary%20rates%20in%20proteins%20are%20better%20modeled%20as%20non-independent%20and%20strictly%20relative&amp;author=AD.%20Fernandes&amp;author=WR.%20Atchley&amp;journal=Bioinformatics&amp;volume=24&amp;issue=19&amp;pages=2177-2183&amp;publication_year=2008&amp;doi=10.1093%2Fbioinformatics%2Fbtn395"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection">Copyright</h2><div class="js-CollapseSection"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;Huang et al.; licensee BioMed Central Ltd.&nbsp;2014<div class="CopyrightComment">
                <p xmlns="" class="SimplePara">This article is published under license to BioMed Central Ltd. This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/2.0"><span class="RefSource">http://creativecommons.org/licenses/by/2.0</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly credited. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div></div></div></section></div>
                            


                        </div>
                    </div>
                    <div class="layout__sidebar-right" id="main-sidebar">
                        
                        <div class="block">
                            <aside role="complementary">
                                <div class="SideBox">
                                    
    <p class="SideBox_action">
        <a class="SideBox_pdf" id="articlePdf" target="_blank" href="/track/pdf/10.1186/1471-2148-14-78?site&#x3D;bmcevolbiol.biomedcentral.com" data-track-event="click" data-event-category="PDF Download"
           data-event-action="Click download article PDF"
           data-event-label="10.1186/1471-2148-14-78">Download PDF</a>
    </p>

                                    
                                    
    <h3 class="export-citations-button" data-toggle="collapse" data-target="js-exportCitations">Export citations</h3>
    <div id="js-exportCitations" class="export-citations-content panel">
        <div class="panel__section">
            <h4 class="panel__title">Citations & References</h4>
            <ul class="ListDownload" id="articleReferences">
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references"
                       data-event-label="10.1186/1471-2148-14-78"
                       href="http://citation-needed.springer.com/v2/references/10.1186/1471-2148-14-78?format=refman&flavour=full">Papers, Zotero, Reference Manager, RefWorks <span>(.RIS)</span></a>
                </li>
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download EndNote citations and references" data-event-label="10.1186/1471-2148-14-78"
                       href="http://citation-needed.springer.com/v2/references/10.1186/1471-2148-14-78?format=endnote&flavour=full">EndNote <span>(.ENW)</span></a></li>
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download Mendeley, JabRef citations and references" data-event-label="10.1186/1471-2148-14-78"
                       href="http://citation-needed.springer.com/v2/references/10.1186/1471-2148-14-78?format=bibtex&flavour=full">Mendeley, JabRef <span>(.BIB)</span></a></li>
            </ul>
        </div>
        <div class="panel__section">
            <h4 class="panel__title">Article citation</h4>
            <ul class="ListDownload">
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-event-label="10.1186/1471-2148-14-78"
                       href="http://citation-needed.springer.com/v2/references/10.1186/1471-2148-14-78?format=refman&flavour=citation">Papers, Zotero, Reference Manager, RefWorks <span>(.RIS)</span></a></li>
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download EndNote article citations" data-event-label="10.1186/1471-2148-14-78"
                       href="http://citation-needed.springer.com/v2/references/10.1186/1471-2148-14-78?format=endnote&flavour=citation">EndNote <span>(.ENW)</span></a></li>
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download Mendeley, JabRef article citations" data-event-label="10.1186/1471-2148-14-78"
                       href="http://citation-needed.springer.com/v2/references/10.1186/1471-2148-14-78?format=bibtex&flavour=citation">Mendeley, JabRef <span>(.BIB)</span></a></li>
            </ul>
        </div>
        <div class="panel__section">
            <h4 class="panel__title">References</h4>
            <ul class="ListDownload">
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-event-label="10.1186/1471-2148-14-78"
                       href="http://citation-needed.springer.com/v2/references/10.1186/1471-2148-14-78?format=refman&flavour=references">Papers, Zotero, Reference Manager, RefWorks <span>(.RIS)</span></a>
                </li>
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download EndNote references" data-event-label="10.1186/1471-2148-14-78"
                       href="http://citation-needed.springer.com/v2/references/10.1186/1471-2148-14-78?format=endnote&flavour=references">EndNote <span>(.ENW)</span></a></li>
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download Mendeley, JabRef references" data-event-label="10.1186/1471-2148-14-78"
                       href="http://citation-needed.springer.com/v2/references/10.1186/1471-2148-14-78?format=bibtex&flavour=references">Mendeley, JabRef <span>(.BIB)</span></a></li>
            </ul>
        </div>
    </div>

                                </div>
                                <div class="SideBox">
                                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" class="table-of-contents" aria-label="table of contents"><div class="table-of-contents__content" data-sticky="toc"><h3 class="table-of-contents__title is-active" data-toggle="collapse" data-target="tocMenu" data-event-category="Table of Contents" data-track-event="click" data-event-action="Toggle ToC" data-event-label="Toggle">Table of Contents</h3><ul class="table-of-contents__menu is-open" role="menu" id="tocMenu"><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Abs1" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Abstract">Abstract</a></li><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Sec1" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Background">Background</a></li><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Sec2" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Methods">Methods</a></li><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Sec9" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Results and discussion">Results and discussion</a></li><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Sec14" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Conclusion">Conclusion</a></li><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Declarations" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Declarations">
                        Declarations
                      </a></li><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Bib1" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="References">References</a></li><li role="menuitem" class="table-of-contents__item is-hidden"><a class="table-of-contents__link" href="#comments" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Comments">Comments</a></li></ul></div></nav>
                                </div>
                                <div class="SideBox">
                                    <h3 class="SideBox_title">Metrics</h3>
                                    <ul class="SideBox_list list-stacked list-stacked--small list-stacked--2">
                                        <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/1471-2148-14-78 #accessCount-container"></div>
                                        <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/1471-2148-14-78 #count-container"></div>
                                        <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/1471-2148-14-78 #altmetric-container"></div>
                                    </ul>
                                </div>
                                
    <div id="sociaMediaShare" class="SideBox">
        <h3 class="SideBox_title">Share this article</h3>
        <ul id="shareButtons" class="share">
            <li class="share__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter share__button share__button--twitter" data-track-event="click"
                        data-event-category="Article Page" data-event-action="Share Twitter" data-event-label="10.1186/1471-2148-14-78">
                    <span class="sr-only">Share on Twitter</span>
                </button>
            </li>
            <li class="share__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook share__button share__button--facebook" data-track-event="click"
                        data-event-category="Article Page" data-event-action="Share Facebook" data-event-label="10.1186/1471-2148-14-78">
                    <span class="sr-only">Share on Facebook</span>
                </button>
            </li>
            <li class="share__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn share__button share__button--linkedin" data-track-event="click"
                        data-event-category="Article Page" data-event-action="Share LinkedIn" data-event-label="10.1186/1471-2148-14-78">
                    <span class="sr-only">Share on LinkedIn</span>
                </button>
            </li>
            <li class="share__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo share__button share__button--weibo" data-track-event="click"
                        data-event-category="Article Page" data-event-action="Share Weibo" data-event-label="10.1186/1471-2148-14-78">
                    <span class="sr-only">Share on Weibo</span>
                </button>
            </li>
            <li class="share__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus share__button share__button--gplus" data-track-event="click"
                        data-event-category="Article Page" data-event-action="Share Google+" data-event-label="10.1186/1471-2148-14-78">
                    <span class="sr-only">Share on Google Plus</span>
                </button>
            </li>
            <li class="share__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit share__button share__button--reddit" data-track-event="click"
                        data-event-category="Article Page" data-event-action="Share Reddit" data-event-label="10.1186/1471-2148-14-78">
                    <span class="sr-only">Share on Reddit</span>
                </button>
            </li>
        </ul>
    </div>

                                
    

                                
    
    <div id="articleSections" class="SideBox">
        <h3 class="SideBox_title">Section</h3>
    <ul class="list-stacked list-stacked--2 list-stacked--small">
        
        <li class="list-stacked__item">
            <a class="list-stacked__link" href="/articles/sections/theories-and-models"
                   data-track-event="click"
                   data-event-category="Article Page"
                   data-event-action="Click section link"
                   data-event-label="10.1186/1471-2148-14-78">Theories and models</a>
            </li>
    
        </ul>
    </div>
    

                                
    

                                <div class="SideBox">
                                    <h3 class="SideBox_title">Other Actions</h3>
                                    <ul class="list-stacked list-stacked--2 list-stacked--small">
                                        <li class="list-stacked__item">
                                            
    <a data-track-event="click" data-event-category="Article Page" data-event-action="Order article reprints" data-event-label="10.1186/1471-2148-14-78" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2F1471-2148-14-78&volume=14&issue=1&title=A+mechanistic+stress+model+of+protein+evolution+accounts+for+site-specific+evolutionary+rates+and+their+relationship+with+packing+density+and+flexibility&author_name=Tsun-Tsao+Huang&start_page=1&end_page=10">Order reprint</a>

                                        </li>
                                    </ul>
                                </div>
                                
        
        <div id="sociaMediaAccounts" class="SideBox">
            <h3 class="SideBox_title">Follow</h3>
            <ul class="follow follow--inline">
                
                <li class="follow__item">
                    <a class="follow__link follow__link--twitter-2" href="https://twitter.com/BMC_series" data-track-event="click" data-event-category="Social Media"
                       data-event-action="Click Follow Social Media" data-event-label="Twitter">
                        <span class="follow__label">Follow us on Twitter</span>
                    </a>
                </li>
                

                

                
            </ul>
        </div>
        
    
                            </aside>
                            
    <div id="FulltextAd" class="adsbox Ad Ad--skyscraper">
        <div class="Ad_content">
            <p class="Ad_label">Advertisement</p>
            <span id="pbgrd-43415139"></span>
        </div>
    </div>

                        </div>
                    </div>
                </div>
            </div>
        </main>
        
            <div class="journal-footer">
                <div class="journal-footer__inner">
                    <div class="journal-footer__summary">
                        <h4 class="journal-footer__title">
                            
                            <span class="journal-footer__title-text">BMC Evolutionary Biology</span>
                        </h4>
                        <p class="journal-footer__issn">ISSN: 1471-2148</p>
                    </div>
                    
                        <div class="journal-footer__contact">
                            <h4 class="journal-footer__contact-title">Contact us</h4>
                            <ul class="journal-footer__contact-list">
                                
                                    <li class="journal-footer__contact-item">Editorial email: <a href="mailto:bmcevolbiol@biomedcentral.com">bmcevolbiol@biomedcentral.com</a></li>
                                
                                
                                    <li class="journal-footer__contact-item">Support email: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
         
            <footer>
                <div class="footer">
                    <div class="footer__header">
                        <img srcset="/static/images/biomed-central-logo-white-3c6c941205.svg" src="/static/images/biomed-central-logo-white-faac4202d5.png" class="logo__img footer__logo" alt="Biomed Central" />
                        
                            <h2 class="footer__menu-title">Publisher Main Menu</h2>
                            <ul class="footer__primary-menu">
                                
                                    <li class="footer__primary-item">
                                        <a class="footer__link footer__primary-link" href="//www.biomedcentral.com/journals">Explore journals</a>
                                    </li>
                                
                                    <li class="footer__primary-item">
                                        <a class="footer__link footer__primary-link" href="//www.biomedcentral.com/getpublished">Get published</a>
                                    </li>
                                
                                    <li class="footer__primary-item">
                                        <a class="footer__link footer__primary-link" href="//www.biomedcentral.com/about">About BioMed Central</a>
                                    </li>
                                
                            </ul>
                        
                    </div>
                    <div id="Test-FooterExists" class="footer__content">
                        <div class="footer__legal">
                            
                            <p class="footer__terms">By continuing to use this website, you agree to our <a class="footer__link footer__link--inline" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>, <a class="footer__link footer__link--inline"
                                    href="//www.biomedcentral.com/terms-and-conditions/privacy-statement">Privacy
                                statement</a> and <a class="footer__link footer__link--inline" href="//www.biomedcentral.com/about/cookies">Cookies</a> policy.</p>
                        </div>
                        
    <nav id="publisherFooter" class="footer__nav">
        <h3 class="footer__menu-title">Publisher secondary menu</h3>
        <div class="footer__nav-inner">
            
            <div class="footer__menu">
                <ul class="footer__list">
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/about/jobs">Jobs</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/login">Receive BioMed Central newsletters</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="https://biomedcentral.typeform.com/to/ve8NOd">Leave feedback</a>
                        
                    </li>
                    
                </ul>
            </div>
            
            <div class="footer__menu">
                <ul class="footer__list">
                    
                    <li class="footer__item">
                        <a class="footer__link" href="http://authorservices.springernature.com/language-editing/">Language editing for authors</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                        
                    </li>
                    
                </ul>
            </div>
            
            <div class="footer__menu">
                <ul class="footer__list">
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/about/policies">Policies</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/licensing">Licensing</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/terms-and-conditions">Terms and conditions</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/terms-and-conditions/privacy-statement">Privacy statement</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/about/cookies">Cookies</a>
                        
                    </li>
                    
                </ul>
            </div>
            

            <div class="footer__menu footer__menu--follow">
                <div class="footer__follow">
                    <h3 class="footer__follow-title">Follow BioMed Central</h3>
                    <ul class="follow follow--inline">
                        
                        <li class="footer__follow-item follow__item">
                            <a class="follow__link follow__link--twitter" href="https://twitter.com/biomedcentral" target="_blank" data-track-event="click" data-event-category="Social Media" data-event-action="Click Follow Social Media" data-event-label="Twitter">
                                <span class="follow__label">Twitter</span>
                            </a>
                        </li>
                        
                        
                        <li class="footer__follow-item follow__item">
                            <a class="follow__link follow__link--facebook" href="https://www.facebook.com/BioMedCentral" target="_blank" data-track-event="click" data-event-category="Social Media" data-event-action="Click Follow Social Media" data-event-label="Facebook">
                                <span class="follow__label">Facebook</span>
                            </a>
                        </li>
                        
                        
                        <li class="footer__follow-item follow__item">
                            <a class="follow__link follow__link--gplus" href="https://plus.google.com/106226337984152901734/" target="_blank" data-track-event="click" data-event-category="Social Media" data-event-action="Click Follow Social Media"
                               data-event-label="Google Plus">
                                <span class="follow__label">Google Plus</span>
                            </a>
                        </li>
                        
                        
                        <li class="footer__follow-item follow__item">
                            <a class="follow__link follow__link--youtube" href="https://www.youtube.com/user/BioMedCentral" target="_blank" data-track-event="click" data-event-category="Social Media" data-event-action="Click Follow Social Media" data-event-label="YouTube">
                                <span class="follow__label">YouTube</span>
                            </a>
                        </li>
                        
                        
                        <li class="footer__follow-item follow__item">
                            <a class="follow__link follow__link--linkedin" href="https://www.linkedin.com/company/biomed-central" target="_blank" data-track-event="click" data-event-category="Social Media" data-event-action="Click Follow Social Media" data-event-label="LinkedIn">
                                <span class="follow__label">LinkedIn</span>
                            </a>
                        </li>
                        
                        
                        <li class="footer__follow-item follow__item">
                            <a class="follow__link follow__link--reddit" href="https://www.reddit.com/domain/biomedcentral.com" target="_blank" data-track-event="click" data-event-category="Social Media" data-event-action="Click Follow Social Media" data-event-label="Reddit">
                                <span class="follow__label">Reddit</span>
                            </a>
                        </li>
                        
                        
                        <li class="footer__follow-item follow__item">
                            <a class="follow__link follow__link--weibo" href="http://www.weibo.com/biomedcentral" target="_blank" data-track-event="click" data-event-category="Social Media" data-event-action="Click Follow Social Media" data-event-label="Weibo">
                                <span class="follow__label">Weibo</span>
                            </a>
                        </li>
                        
                    </ul>
                </div>
            </div>
        </div>
    </nav>


                    </div>
                </div>
                <div class="branding-bar">
                    <div class="logo branding-bar__logo">
                        <img srcset="/static/images/logo-springernature-06a514799d.svg" src="/static/images/logo-springernature-7c95d1378e.png" class="logo__img branding-bar-logo__img" alt="Springer Nature Logo" />
                    </div>
                    <p class="branding-bar__copyright">&#169; 2017 BioMed Central Ltd unless otherwise stated. Part of <a class="branding-bar__link" href="http://www.springernature.com">Springer Nature</a>.</p>
                </div>
            </footer>
            <noscript class="js-cookie-banner-noscript">
        <div class="cookie-banner js-cookie-banner">
            <div class="cookie-banner__inner">
                <p class="cookie-banner__message">
                    We use cookies to improve your experience with our site.
                    <a href="http://www.biomedcentral.com/about/cookies"
                       target="_blank"
                       class="cookie-banner__link"
                       data-track-event="click"
                       data-event-category="EU Cookie Law Banner"
                       data-event-action="Find out more">More information
                    </a>
                </p>
                <button hidden class="cookie-banner__dismiss js-cookie-banner-dismiss"
                        data-track-event="click"
                        data-event-category="EU Cookie Law Banner"
                        data-event-action="Close"><span class="cookie-banner__dismiss-text">Close</span>
                </button>
            </div>
        </div>
</noscript>

        
    </div>
    
    <img class='tracker' style='display:none' src='/track/article/10.1186/1471-2148-14-78' alt=""/>

    <div id="JFulltext"></div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>

    

    
<script>
(function() {
    var linkEl = document.querySelector('.js-ctm');
    if (linkEl && window.matchMedia && window.matchMedia(linkEl.media)) {
        var vendorScript = document.createElement('script');
        vendorScript.src = '/static/js/oscar-vendor-7d446c8d12.js';
        vendorScript.async = false;
        document.body.appendChild(vendorScript);
        var appScript = document.createElement('script');
        appScript.src = '/static/js/oscar-app-1e7a64a0f0.js';
        appScript.async = false;
        document.body.appendChild(appScript);
    }
})();
</script>



<script type="text/javascript">
    if (config) {
        (function (d) {
            Array.prototype.forEach.call(d.getElementsByClassName('RealMediaAd'), function (adElement) {
                var iframe = d.createElement('iframe');
                iframe.frameBorder = '0';
                iframe.scrolling = 'no';
                var position = adElement.getAttribute("data-position"),
                        iframeInDoc = adElement.getElementsByClassName('Ad_content')[0].appendChild(iframe),
                        doc = iframeInDoc.contentWindow.document;

                doc.open().write('<body onload="window.location.href = '
                        + '\'/placeholder/v1/advertising/realMedia?siteWithPath=' + encodeURIComponent(config.siteWithPath)
                        + '&placement=' + position + '\';">');
                doc.close();
            });
        })(document);
    }
</script>



    
        <script type="text/javascript">
            document.addEventListener('readystatechange', function () {
                if (document.readyState === 'complete') {
                    setTimeout(function () {
                        var script = document.createElement("script");
                        script.type = "text/javascript";
                        script.src = "https://recommendations.springernature.com/latest/entry-point.js";
                        document.getElementsByTagName("head")[0].appendChild(script);
                    }, 0);
                }
            });
        </script>
    


    <script type="text/javascript">
        var _kiq = _kiq || [];
        (function () {
            setTimeout(function () {
                var d = document, f = d.getElementsByTagName('script')[0], s = d.createElement('script');
                s.type = 'text/javascript';
                s.async = true;
                s.src = '//s3.amazonaws.com/ki.js/47412/9CC.js';
                f.parentNode.insertBefore(s, f);
            }, 1);
            setTimeout(function () {
                var d = document, f = d.getElementsByTagName('script')[0], s = d.createElement('script');
                s.type = 'text/javascript';
                s.async = true;
                s.src = '//s3.amazonaws.com/ki.js/47412/aC6.js';
                f.parentNode.insertBefore(s, f);
            }, 1);
        })();
    </script>


<noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
</body>
</html>
